# Appendix XIV: Glossary of schema objects

```{glossary}

.* (extensions)

	name: Any Extension

	description:
	>Any extension is allowed.

.ave (extensions)

	name: AVE

	description:
	>File containing data averaged by segments of interest.<br>Used by KIT, Yokogawa, and Ricoh MEG systems.

.bdf (extensions)

	name: Biosemi Data Format

	description:
	>A [Biosemi](https://www.biosemi.com/) Data Format file.<br>Each recording consists of a single `.bdf` file. [`bdf+`](https://www.teuniz.net/edfbrowser/bdfplus%20format%20description.html) files are permitted. The capital `.BDF` extension MUST NOT be used.

.bval (extensions)

	name: FSL-Format Gradient Amplitudes

	description:
	>A space-delimited file containing gradient directions (b-vectors) of diffusion measurement.<br>The `bval` file contains the *b*-values (in s/mm<sup>2</sup>) corresponding to the volumes in the relevant NIfTI file, with 0 designating *b*=0 volumes.

.bvec (extensions)

	name: FSL-Format Gradient Directions

	description:
	>A space-delimited file containing gradient directions (b-vectors) of diffusion measurement.<br>This file contains 3 rows with *N* space-delimited floating-point numbers, corresponding to the *N* volumes in the corresponding NIfTI file.<br>The first row contains the *x* elements, the second row contains the *y* elements and the third row contains the *z* elements of a unit vector in the direction of the applied diffusion gradient, where the *i*-th elements in each row correspond together to the *i*-th volume, with `[0,0,0]` for *non-diffusion-weighted* (also called *b*=0 or *low-b*) volumes.<br>Following the FSL format for the `bvec` specification, the coordinate system of the *b* vectors MUST be defined with respect to the coordinate system defined by the header of the corresponding `_dwi` NIfTI file and not the scanner's device coordinate system (see [Coordinate systems](../99-appendices/08-coordinate-systems.md)). The most relevant limitation imposed by this choice is that the gradient information cannot be directly stored in this format if the scanner generates *b*-vectors in *scanner coordinates*.

.chn (extensions)

	name: KRISS CHN

	description:
	>A file generated by KRISS MEG systems containing the position of the center of the MEG coils.<br>Each experimental run on the KRISS system produces a file with extension `.kdf`. Additional files that may be available in the same folder include the digitized positions of the head points (`\_digitizer.txt`), the position of the center of the MEG coils (`.chn`), and the event markers (`.trg`).

.con (extensions)

	name: KIT/Yokogawa/Ricoh Continuous Data

	description:
	>Raw continuous data from a KIT/Yokogawa/Ricoh MEG system.<br>Successor to the `.sqd` extension for raw continuous data.

.dat (extensions)

	name: MEG Fine-Calibration Format

	description:
	>A fine-calibration file used for Neuromag/Elekta/MEGIN MEG recording hardware.

.ds/ (extensions)

	name: CTF MEG Dataset Folder

	description:
	>A folder for MEG data, typically containing a `.meg4` file for the data and a `.res4` file for the resources.

.edf (extensions)

	name: European Data Format

	description:
	>A [European data format](https://www.edfplus.info/) file.<br>Each recording consists of a single `.edf`` file. [`edf+`](https://www.edfplus.info/specs/edfplus.html) files are permitted. The capital `.EDF` extension MUST NOT be used.

.eeg (extensions)

	name: BrainVision Binary Data

	description:
	>A binary data file in the [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5). These files come in three-file sets, including a `.vhdr`, a `.vmrk`, and a `.eeg` file.

.fdt (extensions)

	name: EEGLAB FDT

	description:
	>An [EEGLAB](https://sccn.ucsd.edu/eeglab) file.<br>The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab). Each recording consists of a `.set` file with an optional `.fdt` file.

.fif (extensions)

	name: Functional Imaging File Format

	description:
	>An MEG file format used by Neuromag, Elekta, and MEGIN.

.jpg (extensions)

	name: Joint Photographic Experts Group Format

	description:
	>A JPEG image file.

.json (extensions)

	name: JavaScript Object Notation

	description:
	>A JSON file.<br>In the BIDS specification, JSON files are primarily used as "sidecar" files, in which metadata describing "data" files are encoded. These sidecar files follow the inheritance principle.<br>There are also a few special cases of JSON files being first-order data files, such as `genetic_info.json`.

.kdf (extensions)

	name: KRISS KDF

	description:
	>A KRISS (file with extension `.kdf`) file.<br>Each experimental run on the KRISS system produces a file with extension `.kdf`. Additional files that may be available in the same folder include the digitized positions of the head points (`\_digitizer.txt`), the position of the center of the MEG coils (`.chn`), and the event markers (`.trg`).

.md (extensions)

	name: Markdown

	description:
	>A Markdown file.

.mefd/ (extensions)

	name: Multiscale Electrophysiology File Format Version 3.0

	description:
	>A folder in the [MEF3](https://osf.io/e3sf9/) format.<br>Each recording consists of a `.mefd` directory.

.mhd (extensions)

	name: ITAB Binary Header

	description:
	>Produced by ITAB-ARGOS153 systems. This file a binary header file, and is generated along with a raw data file with the `.raw` extension.

.mrk (extensions)

	name: MRK

	description:
	>A file containing MEG sensor coil positions.<br>Used by KIT, Yokogawa, and Ricoh MEG systems. Successor to the `.sqd` extension for marker files.

.nii (extensions)

	name: NIfTI

	description:
	>A Neuroimaging Informatics Technology Initiative (NIfTI) data file.

.nii.gz (extensions)

	name: Compressed NIfTI

	description:
	>A compressed Neuroimaging Informatics Technology Initiative (NIfTI) data file.

.nwb (extensions)

	name: Neurodata Without Borders Format

	description:
	>A [Neurodata Without Borders](https://nwb-schema.readthedocs.io) file.<br>Each recording consists of a single `.nwb` file.

.ome.btf (extensions)

	name: Open Microscopy Environment BigTIFF

	description:
	>A [BigTIFF](https://www.awaresystems.be/imaging/tiff/bigtiff.html) image file, for very large images.

.ome.tif (extensions)

	name: Open Microscopy Environment Tag Image File Format

	description:
	>An [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/specification.html#) image file.

.png (extensions)

	name: Portable Network Graphics

	description:
	>A [Portable Network Graphics](http://www.libpng.org/pub/png/) file.

.pos (extensions)

	name: Head Point Position

	description:
	>File containing digitized positions of the head points.<br>This may be produced by a 4D neuroimaging/BTi MEG system or a CTF MEG system.

.raw (extensions)

	name: RAW

	description:
	>When produced by a KIT / Yokogawa / Ricoh MEG system, this file contains trial-based evoked fields.<br>When produced by an ITAB-ARGOS153 system, this file contains raw data and is generated along with an associated binary header file  with the `.mhd` extension.

.rst (extensions)

	name: reStructuredText

	description:
	>A [reStructuredText](https://docutils.sourceforge.io/rst.html) file.

.set (extensions)

	name: EEGLAB SET

	description:
	>An [EEGLAB](https://sccn.ucsd.edu/eeglab) file.<br>The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab). Each recording consists of a `.set` file with an optional `.fdt` file.

.sqd (extensions)

	name: SQD

	description:
	>A file containing either raw MEG data or MEG sensor coil positions. While this extension is still valid, it has been succeeded by `.con` for raw MEG data and `.mrk` for marker information.<br>Used by KIT, Yokogawa, and Ricoh MEG systems.

.tif (extensions)

	name: Tag Image File Format

	description:
	>A [Tag Image File Format](https://en.wikipedia.org/wiki/TIFF) file.

.trg (extensions)

	name: KRISS TRG

	description:
	>A file generated by KRISS MEG systems containing the event markers.<br>Each experimental run on the KRISS system produces a file with extension `.kdf`. Additional files that may be available in the same folder include the digitized positions of the head points (`\_digitizer.txt`), the position of the center of the MEG coils (`.chn`), and the event markers (`.trg`).

.tsv (extensions)

	name: Tab-Delimited

	description:
	>A tab-delimited file.

.tsv.gz (extensions)

	name: Compressed Tab-Delimited

	description:
	>A gzipped tab-delimited file. This file extension is only used for very large tabular data, such as physiological recordings. For smaller data, the unzipped `.tsv` extension is preferred.

.txt (extensions)

	name: Text

	description:
	>A free-form text file.<br>Tab-delimited files should have the `.tsv` extension rather than a `.txt` extension.

.vhdr (extensions)

	name: BrainVision Text Header

	description:
	>A text header file in the [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5). These files come in three-file sets, including a `.vhdr`, a `.vmrk`, and a `.eeg` file.

.vmrk (extensions)

	name: BrainVision Marker

	description:
	>A text marker file in the [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5). These files come in three-file sets, including a `.vhdr`, a `.vmrk`, and a `.eeg` file.

/ (extensions)

	name: Folder

	description:
	>A folder with no extension. Corresponds to BTi/4D data.

2PE (suffixes)

	name: 2-photon excitation microscopy

	description:
	>2-photon excitation microscopy imaging data

Acknowledgements (metadata)

	name: Acknowledgements

	description:
	>Text acknowledging contributions of individuals or institutions beyond those listed in Authors or Funding.

	schema information:
	```yaml
	{'type': 'string'}
	```

AcquisitionDuration (metadata)

	name: AcquisitionDuration

	description:
	>Duration (in seconds) of volume acquisition. Corresponds to DICOM Tag 0018, 9073 `Acquisition Duration`. This field is mutually exclusive with `"RepetitionTime"`.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'unit': 's'}
	```

AcquisitionMode (metadata)

	name: AcquisitionMode

	description:
	>Type of acquisition of the PET data (for example, `"list mode"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

AcquisitionVoxelSize (metadata)

	name: AcquisitionVoxelSize

	description:
	>An array of numbers with a length of 3, in millimeters. This parameter denotes the original acquisition voxel size, excluding any inter-slice gaps and before any interpolation or resampling within reconstruction or image processing. Any point spread function effects, for example due to T2-blurring, that would decrease the effective resolution are not considered here.

	schema information:
	```yaml
	{'type': 'array', 'minItems': 3, 'maxItems': 3, 'items': {'type': 'number', 'exclusiveMinimum': 0, 'unit': 'mm'}}
	```

Anaesthesia (metadata)

	name: Anaesthesia

	description:
	>Details of anaesthesia used, if any.

	schema information:
	```yaml
	{'type': 'string'}
	```

AnalyticalApproach (metadata)

	name: AnalyticalApproach

	description:
	>Methodology or methodologies used to analyse the `"GeneticLevel"`. Values MUST be taken from the [database of Genotypes and Phenotypes (dbGaP)](https://www.ncbi.nlm.nih.gov/gap/advanced) under /Study/Molecular Data Type (for example, SNP Genotypes (Array) or Methylation (CpG).

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

AnatomicalLandmarkCoordinateSystem (metadata)

	name: AnatomicalLandmarkCoordinateSystem

	description:
	>Defines the coordinate system for the anatomical landmarks. See [Appendix VIII](/99-appendices/08-coordinate-systems.html) for a list of restricted keywords for coordinate systems. If `"Other"`, provide definition of the coordinate system in `"AnatomicalLandmarkCoordinateSystemDescription"`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['CTF', 'ElektaNeuromag', '4DBti', 'KitYokogawa', 'ChietiItab', 'Other', 'CapTrak', 'EEGLAB', 'EEGLAB-HJ', 'Other', 'ICBM452AirSpace', 'ICBM452Warp5Space', 'IXI549Space', 'fsaverage', 'fsaverageSym', 'fsLR', 'MNIColin27', 'MNI152Lin', 'MNI152NLin2009aSym', 'MNI152NLin2009bSym', 'MNI152NLin2009cSym', 'MNI152NLin2009aAsym', 'MNI152NLin2009bAsym', 'MNI152NLin2009cAsym', 'MNI152NLin6Sym', 'MNI152NLin6ASym', 'MNI305', 'NIHPD', 'OASIS30AntsOASISAnts', 'OASIS30Atropos', 'Talairach', 'UNCInfant', 'fsaverage3', 'fsaverage4', 'fsaverage5', 'fsaverage6', 'fsaveragesym', 'UNCInfant0V21', 'UNCInfant1V21', 'UNCInfant2V21', 'UNCInfant0V22', 'UNCInfant1V22', 'UNCInfant2V22', 'UNCInfant0V23', 'UNCInfant1V23', 'UNCInfant2V23']}
	```

AnatomicalLandmarkCoordinateSystemDescription (metadata)

	name: AnatomicalLandmarkCoordinateSystemDescription

	description:
	>Free-form text description of the coordinate system. May also include a link to a documentation page or paper describing the system in greater detail.

	schema information:
	```yaml
	{'type': 'string'}
	```

AnatomicalLandmarkCoordinateUnits (metadata)

	name: AnatomicalLandmarkCoordinateUnits

	description:
	>Units of the coordinates of `"AnatomicalLandmarkCoordinateSystem"`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['m', 'mm', 'cm', 'n/a']}
	```

AnatomicalLandmarkCoordinates _sense 1_ (metadata)

	name: AnatomicalLandmarkCoordinates

	description:
	>Key:value pairs of the labels and 3-D digitized locations of anatomical landmarks, interpreted following the `"AnatomicalLandmarkCoordinateSystem"` (for example, `{"NAS": [12.7,21.3,13.9], "LPA": [5.2,11.3,9.6], "RPA": [20.2,11.3,9.1]}`. Each array MUST contain three numeric values corresponding to x, y, and z axis of the coordinate system in that exact order.

	schema information:
	```yaml
	{'type': 'object', 'additionalProperties': {'type': 'array', 'items': {'type': 'number'}, 'minItems': 3, 'maxItems': 3}}
	```

AnatomicalLandmarkCoordinates _sense 2_ (metadata)

	name: AnatomicalLandmarkCoordinates

	description:
	>Key:value pairs of any number of additional anatomical landmarks and their coordinates in voxel units (where first voxel has index 0,0,0) relative to the associated anatomical MRI (for example, `{"AC": [127,119,149], "PC": [128,93,141], "IH": [131,114,206]}`, or `{"NAS": [127,213,139], "LPA": [52,113,96], "RPA": [202,113,91]}`). Each array MUST contain three numeric values corresponding to x, y, and z axis of the coordinate system in that exact order.

	schema information:
	```yaml
	{'type': 'object', 'additionalProperties': {'type': 'array', 'items': {'type': 'number'}, 'minItems': 3, 'maxItems': 3}}
	```

ArterialSpinLabelingType (metadata)

	name: ArterialSpinLabelingType

	description:
	>The arterial spin labeling type.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['CASL', 'PCASL', 'PASL']}
	```

AssociatedEmptyRoom (metadata)

	name: AssociatedEmptyRoom

	description:
	>Relative path in BIDS directory structure to empty-room file associated with the subject's MEG recording. The path needs to use forward slashes instead of backward slashes (for example, "sub-emptyroom/ses-/meg/sub-emptyroom_ses-_task-noise_run-_meg.ds").

	schema information:
	```yaml
	{'anyOf': [{'type': 'array', 'items': {'type': 'string', 'format': 'dataset_relative'}}, {'type': 'string', 'format': 'dataset_relative'}]}
	```

Atlas (metadata)

	name: Atlas

	description:
	>Which atlas (if any) was used to generate the mask.

	schema information:
	```yaml
	{'type': 'string'}
	```

AttenuationCorrection (metadata)

	name: AttenuationCorrection

	description:
	>Short description of the attenuation correction method used.

	schema information:
	```yaml
	{'type': 'string'}
	```

AttenuationCorrectionMethodReference (metadata)

	name: AttenuationCorrectionMethodReference

	description:
	>Reference paper for the attenuation correction method used.

	schema information:
	```yaml
	{'type': 'string'}
	```

Authors (metadata)

	name: Authors

	description:
	>List of individuals who contributed to the creation/curation of the dataset.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'string'}}
	```

B0FieldIdentifier (metadata)

	name: B0FieldIdentifier

	description:
	>The presence of this key states that this particular 3D or 4D image MAY be used for fieldmap estimation purposes. Each `"B0FieldIdentifier"` MUST be a unique string within one participant's tree, shared only by the images meant to be used as inputs for the estimation of a particular instance of the *B<sub>0</sub> field* estimation. It is RECOMMENDED to derive this identifier from DICOM Tags, for example, DICOM tag 0018, 1030 `Protocol Name`, or DICOM tag 0018, 0024 `Sequence Name` when the former is not defined (for example, in GE devices.)

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

B0FieldSource (metadata)

	name: B0FieldSource

	description:
	>At least one existing `"B0FieldIdentifier"` defined by images in the participant's tree. This field states the *B<sub>0</sub> field* estimation designated by the `"B0FieldIdentifier"` that may be used to correct the dataset for distortions caused by B<sub>0</sub> inhomogeneities. `"B0FieldSource"` and `"B0FieldIdentifier"` MAY both be present for images that are used to estimate their own B<sub>0</sub> field, for example, in "pepolar" acquisitions.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

BF (suffixes)

	name: Bright-field microscopy

	description:
	>Bright-field microscopy imaging data

BIDSVersion (metadata)

	name: BIDSVersion

	description:
	>The version of the BIDS standard that was used.

	schema information:
	```yaml
	{'type': 'string'}
	```

BackgroundSuppression (metadata)

	name: BackgroundSuppression

	description:
	>Boolean indicating if background suppression is used.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

BackgroundSuppressionNumberPulses (metadata)

	name: BackgroundSuppressionNumberPulses

	description:
	>The number of background suppression pulses used. Note that this excludes any effect of background suppression pulses applied before the labeling.

	schema information:
	```yaml
	{'type': 'number', 'minimum': 0}
	```

BackgroundSuppressionPulseTime (metadata)

	name: BackgroundSuppressionPulseTime

	description:
	>Array of numbers containing timing, in seconds, of the background suppression pulses with respect to the start of the labeling. In case of multi-PLD with different background suppression pulse times, only the pulse time of the first PLD should be defined.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number', 'minimum': 0, 'unit': 's'}}
	```

BasedOn (metadata)

	name: BasedOn

	description:
	>List of files in a file collection to generate the map. Fieldmaps are also listed, if involved in the processing.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string', 'format': 'participant_relative'}, {'type': 'array', 'items': {'type': 'string', 'format': 'participant_relative'}}]}
	```

BloodDensity (metadata)

	name: BloodDensity

	description:
	>Measured blood density. Unit of blood density should be in `"g/mL"`.

	schema information:
	```yaml
	{'type': 'number', 'unit': 'g/mL'}
	```

BodyPart (metadata)

	name: BodyPart

	description:
	>Body part of the organ / body region scanned.

	schema information:
	```yaml
	{'type': 'string'}
	```

BodyPartDetails (metadata)

	name: BodyPartDetails

	description:
	>Additional details about body part or location (for example: `"corpus callosum"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

BodyPartDetailsOntology (metadata)

	name: BodyPartDetailsOntology

	description:
	>[URI](/02-common-principles.html#uniform-resource-indicator) of ontology used for BodyPartDetails (for example: `"https://www.ebi.ac.uk/ols/ontologies/uberon"`).

	schema information:
	```yaml
	{'type': 'string', 'format': 'uri'}
	```

BolusCutOffDelayTime (metadata)

	name: BolusCutOffDelayTime

	description:
	>Duration between the end of the labeling and the start of the bolus cut-off saturation pulse(s), in seconds. This can be a number or array of numbers, of which the values must be non-negative and monotonically increasing, depending on the number of bolus cut-off saturation pulses. For Q2TIPS, only the values for the first and last bolus cut-off saturation pulses are provided. Based on DICOM Tag 0018, 925F `ASL Bolus Cut-off Delay Time`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'minimum': 0, 'unit': 's'}, {'type': 'array', 'items': {'type': 'number', 'unit': 's', 'minimum': 0}}]}
	```

BolusCutOffFlag (metadata)

	name: BolusCutOffFlag

	description:
	>Boolean indicating if a bolus cut-off technique is used. Corresponds to DICOM Tag 0018, 925C `ASL Bolus Cut-off Flag`.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

BolusCutOffTechnique (metadata)

	name: BolusCutOffTechnique

	description:
	>Name of the technique used, for example `"Q2TIPS"`, `"QUIPSS"`, `"QUIPSSII"`. Corresponds to DICOM Tag 0018, 925E `ASL Bolus Cut-off Technique`.

	schema information:
	```yaml
	{'type': 'string'}
	```

BrainLocation (metadata)

	name: BrainLocation

	description:
	>Refers to the location in space of the `"TissueOrigin"`. Values may be an MNI coordinate, a label taken from the [Allen Brain Atlas](https://atlas.brain-map.org/atlas?atlas=265297125&plate=112360888&structure=4392&x=40348.15104166667&y=46928.75&zoom=-7&resolution=206.60&z=3), or layer to refer to layer-specific gene expression, which can also tie up with laminar fMRI.

	schema information:
	```yaml
	{'type': 'string'}
	```

CARS (suffixes)

	name: Coherent anti-Stokes Raman spectroscopy

	description:
	>Coherent anti-Stokes Raman spectroscopy imaging data

CASLType (metadata)

	name: CASLType

	description:
	>Describes if a separate coil is used for labeling.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['single-coil', 'double-coil']}
	```

CHANGES (top_level_files)

	name: CHANGES

	description:
	>Version history of the dataset (describing changes, updates and corrections) MAY be provided in the form of a `CHANGES` text file. This file MUST follow the [CPAN Changelog convention](https://metacpan.org/pod/release/HAARG/CPAN-Changes-0.400002/lib/CPAN/Changes/Spec.pod). The `CHANGES` file MUST be either in ASCII or UTF-8 encoding.

CONF (suffixes)

	name: Confocal microscopy

	description:
	>Confocal microscopy imaging data

CapManufacturer (metadata)

	name: CapManufacturer

	description:
	>Name of the cap manufacturer (for example, `"EasyCap"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

CapManufacturersModelName (metadata)

	name: CapManufacturersModelName

	description:
	>Manufacturer's designation of the EEG cap model (for example, `"actiCAP 64 Ch Standard-2"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

CellType (metadata)

	name: CellType

	description:
	>Describes the type of cell analyzed. Values SHOULD come from the [cell ontology](http://obofoundry.org/ontology/cl.html).

	schema information:
	```yaml
	{'type': 'string'}
	```

Chimap (suffixes)

	name: Quantitative susceptibility map (QSM)

	description:
	>In parts per million (ppm). QSM allows for determining the underlying magnetic susceptibility of tissue (Chi) ([Wang & Liu, 2014](https://doi.org/10.1002/mrm.25358)). Chi maps are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': 'ppm'}
	```

ChunkTransformationMatrix (metadata)

	name: ChunkTransformationMatrix

	description:
	>3x3 or 4x4 affine transformation matrix describing spatial chunk transformation, for 2D and 3D respectively (for examples: `[[2, 0, 0], [0, 3, 0], [0, 0, 1]]` in 2D for 2x and 3x scaling along the first and second axis respectively; or `[[1, 0, 0, 0], [0, 2, 0, 0], [0, 0, 3, 0], [0, 0, 0, 1]]` in 3D for 2x and 3x scaling along the second and third axis respectively). Note that non-spatial dimensions like time and channel are not included in the transformation matrix.

	schema information:
	```yaml
	{'anyOf': [{'type': 'array', 'minItems': 3, 'maxItems': 3, 'items': {'type': 'array', 'minItems': 3, 'maxItems': 3, 'items': {'type': 'number'}}}, {'type': 'array', 'minItems': 4, 'maxItems': 4, 'items': {'type': 'array', 'minItems': 4, 'maxItems': 4, 'items': {'type': 'number'}}}]}
	```

ChunkTransformationMatrixAxis (metadata)

	name: ChunkTransformationMatrixAxis

	description:
	>Describe the axis of the ChunkTransformationMatrix (for examples: `["X", "Y"]` or `["Z", "Y", "X"]`).

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'string', 'minItems': 2, 'maxItems': 3}}
	```

Code (metadata)

	name: Code

	description:
	>[URI](/02-common-principles.html#uniform-resource-indicator) of the code used to present the stimuli. Persistent identifiers such as DOIs are preferred. If multiple versions of code may be hosted at the same location, revision-specific URIs are recommended.

	schema information:
	```yaml
	{'type': 'string', 'format': 'uri'}
	```

CogAtlasID (metadata)

	name: CogAtlasID

	description:
	>[URI](/02-common-principles.html#uniform-resource-indicator) of the corresponding [Cognitive Atlas](https://www.cognitiveatlas.org/) Task term.

	schema information:
	```yaml
	{'type': 'string', 'format': 'uri'}
	```

CogPOID (metadata)

	name: CogPOID

	description:
	>[URI](/02-common-principles.html#uniform-resource-indicator) of the corresponding [CogPO](http://www.cogpo.org/) term.

	schema information:
	```yaml
	{'type': 'string', 'format': 'uri'}
	```

CoilCombinationMethod (metadata)

	name: CoilCombinationMethod

	description:
	>Almost all fMRI studies using phased-array coils use root-sum-of-squares (rSOS) combination, but other methods exist. The image reconstruction is changed by the coil combination method (as for the matrix coil mode above), so anything non-standard should be reported.

	schema information:
	```yaml
	{'type': 'string'}
	```

Columns (metadata)

	name: Columns

	description:
	>Names of columns in file.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'string'}}
	```

ContinuousHeadLocalization (metadata)

	name: ContinuousHeadLocalization

	description:
	>`true` or `false` value indicating whether continuous head localisation was performed.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

ContrastBolusIngredient (metadata)

	name: ContrastBolusIngredient

	description:
	>Active ingredient of agent. Corresponds to DICOM Tag 0018, 1048 `Contrast/Bolus Ingredient`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['IODINE', 'GADOLINIUM', 'CARBON DIOXIDE', 'BARIUM', 'XENON']}
	```

DCOffsetCorrection (metadata)

	name: DCOffsetCorrection

	description:
	>A description of the method (if any) used to correct for a DC offset. If the method used was subtracting the mean value for each channel, use "mean".

	schema information:
	```yaml
	{'type': 'string'}
	```

DF (suffixes)

	name: Dark-field microscopy

	description:
	>Dark-field microscopy imaging data

DIC (suffixes)

	name: Differential interference contrast microscopy

	description:
	>Differential interference contrast microscopy imaging data

DatasetDOI (metadata)

	name: DatasetDOI

	description:
	>The Digital Object Identifier of the dataset (not the corresponding paper). DOIs SHOULD be expressed as a valid [URI](/02-common-principles.html#uniform-resource-indicator); bare DOIs such as `10.0.2.3/dfjj.10` are [DEPRECATED](/02-common-principles.html#definitions).

	schema information:
	```yaml
	{'type': 'string', 'format': 'uri'}
	```

DatasetType (metadata)

	name: DatasetType

	description:
	>The interpretation of the dataset. For backwards compatibility, the default value is `"raw"`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['raw', 'derivative']}
	```

DecayCorrectionFactor (metadata)

	name: DecayCorrectionFactor

	description:
	>Decay correction factor for each frame.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number'}}
	```

DelayAfterTrigger (metadata)

	name: DelayAfterTrigger

	description:
	>Duration (in seconds) from trigger delivery to scan onset. This delay is commonly caused by adjustments and loading times. This specification is entirely independent of `"NumberOfVolumesDiscardedByScanner"` or `"NumberOfVolumesDiscardedByUser"`, as the delay precedes the acquisition.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

DelayTime (metadata)

	name: DelayTime

	description:
	>User specified time (in seconds) to delay the acquisition of data for the following volume. If the field is not present it is assumed to be set to zero. Corresponds to Siemens CSA header field `lDelayTimeInTR`. This field is REQUIRED for sparse sequences using the `"RepetitionTime"` field that do not have the `"SliceTiming"` field set to allowed for accurate calculation of "acquisition time". This field is mutually exclusive with `"VolumeTiming"`.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

Density (metadata)

	name: Density

	description:
	>Specifies the interpretation of the density keyword. If an object is used, then the keys should be values for the ``den`` entity and values should be descriptions of those ``den`` values.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'object', 'additionalProperties': {'type': 'string'}}]}
	```

Derivative (metadata)

	name: Derivative

	description:
	>Indicates that values in the corresponding column are transformations of values from other columns (for example a summary score based on a subset of items in a questionnaire).

	schema information:
	```yaml
	{'type': 'boolean'}
	```

Description (metadata)

	name: Description

	description:
	>Free-form natural language description.

	schema information:
	```yaml
	{'type': 'string'}
	```

DeviceSerialNumber (metadata)

	name: DeviceSerialNumber

	description:
	>The serial number of the equipment that produced the measurements. A pseudonym can also be used to prevent the equipment from being identifiable, so long as each pseudonym is unique within the dataset.

	schema information:
	```yaml
	{'type': 'string'}
	```

DewarPosition (metadata)

	name: DewarPosition

	description:
	>Position of the dewar during the MEG scan: `"upright"`, `"supine"` or `"degrees"` of angle from vertical: for example on CTF systems, `"upright=15°, supine=90°"`.

	schema information:
	```yaml
	{'type': 'string'}
	```

DigitizedHeadPoints (metadata)

	name: DigitizedHeadPoints

	description:
	>`true` or `false` value indicating whether head points outlining the scalp/face surface are contained within this recording.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

DigitizedHeadPointsCoordinateSystem (metadata)

	name: DigitizedHeadPointsCoordinateSystem

	description:
	>Defines the coordinate system for the digitized head points. See [Appendix VIII](/99-appendices/08-coordinate-systems.html) for a list of restricted keywords for coordinate systems. If `"Other"`, provide definition of the coordinate system in `"DigitizedHeadPointsCoordinateSystemDescription"`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['CTF', 'ElektaNeuromag', '4DBti', 'KitYokogawa', 'ChietiItab', 'Other', 'CapTrak', 'EEGLAB', 'EEGLAB-HJ', 'Other', 'ICBM452AirSpace', 'ICBM452Warp5Space', 'IXI549Space', 'fsaverage', 'fsaverageSym', 'fsLR', 'MNIColin27', 'MNI152Lin', 'MNI152NLin2009aSym', 'MNI152NLin2009bSym', 'MNI152NLin2009cSym', 'MNI152NLin2009aAsym', 'MNI152NLin2009bAsym', 'MNI152NLin2009cAsym', 'MNI152NLin6Sym', 'MNI152NLin6ASym', 'MNI305', 'NIHPD', 'OASIS30AntsOASISAnts', 'OASIS30Atropos', 'Talairach', 'UNCInfant', 'fsaverage3', 'fsaverage4', 'fsaverage5', 'fsaverage6', 'fsaveragesym', 'UNCInfant0V21', 'UNCInfant1V21', 'UNCInfant2V21', 'UNCInfant0V22', 'UNCInfant1V22', 'UNCInfant2V22', 'UNCInfant0V23', 'UNCInfant1V23', 'UNCInfant2V23']}
	```

DigitizedHeadPointsCoordinateSystemDescription (metadata)

	name: DigitizedHeadPointsCoordinateSystemDescription

	description:
	>Free-form text description of the coordinate system. May also include a link to a documentation page or paper describing the system in greater detail.

	schema information:
	```yaml
	{'type': 'string'}
	```

DigitizedHeadPointsCoordinateUnits (metadata)

	name: DigitizedHeadPointsCoordinateUnits

	description:
	>Units of the coordinates of `"DigitizedHeadPointsCoordinateSystem"`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['m', 'mm', 'cm', 'n/a']}
	```

DigitizedLandmarks (metadata)

	name: DigitizedLandmarks

	description:
	>`true` or `false` value indicating whether anatomical landmark points (fiducials) are contained within this recording.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

DispersionConstant (metadata)

	name: DispersionConstant

	description:
	>External dispersion time constant resulting from tubing in default unit seconds.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

DispersionCorrected (metadata)

	name: DispersionCorrected

	description:
	>Boolean flag specifying whether the blood data have been dispersion-corrected. NOTE: not customary for manual samples, and hence should be set to `false`.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

DoseCalibrationFactor (metadata)

	name: DoseCalibrationFactor

	description:
	>Multiplication factor used to transform raw data (in counts/sec) to meaningful unit (Bq/ml). Corresponds to DICOM Tag 0054, 1322 `Dose Calibration Factor`.

	schema information:
	```yaml
	{'type': 'number'}
	```

DwellTime (metadata)

	name: DwellTime

	description:
	>Actual dwell time (in seconds) of the receiver per point in the readout direction, including any oversampling. For Siemens, this corresponds to DICOM field 0019, 1018 (in ns). This value is necessary for the optional readout distortion correction of anatomicals in the HCP Pipelines. It also usefully provides a handle on the readout bandwidth, which isn't captured in the other metadata tags. Not to be confused with `"EffectiveEchoSpacing"`, and the frequent mislabeling of echo spacing (which is spacing in the phase encoding direction) as "dwell time" (which is spacing in the readout direction).

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

ECGChannelCount (metadata)

	name: ECGChannelCount

	description:
	>Number of ECG channels.

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

ECOGChannelCount (metadata)

	name: ECOGChannelCount

	description:
	>Number of ECoG channels.

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

EEGChannelCount (metadata)

	name: EEGChannelCount

	description:
	>Number of EEG channels recorded simultaneously (for example, `21`).

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

EEGCoordinateSystem (metadata)

	name: EEGCoordinateSystem

	description:
	>Defines the coordinate system for the EEG sensors.<br>See [Appendix VIII](/99-appendices/08-coordinate-systems.html) for a list of restricted keywords for coordinate systems. If `"Other"`, provide definition of the coordinate system in `EEGCoordinateSystemDescription`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['CTF', 'ElektaNeuromag', '4DBti', 'KitYokogawa', 'ChietiItab', 'Other', 'CapTrak', 'EEGLAB', 'EEGLAB-HJ', 'Other', 'ICBM452AirSpace', 'ICBM452Warp5Space', 'IXI549Space', 'fsaverage', 'fsaverageSym', 'fsLR', 'MNIColin27', 'MNI152Lin', 'MNI152NLin2009aSym', 'MNI152NLin2009bSym', 'MNI152NLin2009cSym', 'MNI152NLin2009aAsym', 'MNI152NLin2009bAsym', 'MNI152NLin2009cAsym', 'MNI152NLin6Sym', 'MNI152NLin6ASym', 'MNI305', 'NIHPD', 'OASIS30AntsOASISAnts', 'OASIS30Atropos', 'Talairach', 'UNCInfant', 'fsaverage3', 'fsaverage4', 'fsaverage5', 'fsaverage6', 'fsaveragesym', 'UNCInfant0V21', 'UNCInfant1V21', 'UNCInfant2V21', 'UNCInfant0V22', 'UNCInfant1V22', 'UNCInfant2V22', 'UNCInfant0V23', 'UNCInfant1V23', 'UNCInfant2V23']}
	```

EEGCoordinateSystemDescription (metadata)

	name: EEGCoordinateSystemDescription

	description:
	>Free-form text description of the coordinate system. May also include a link to a documentation page or paper describing the system in greater detail.

	schema information:
	```yaml
	{'type': 'string'}
	```

EEGCoordinateUnits (metadata)

	name: EEGCoordinateUnits

	description:
	>Units of the coordinates of `EEGCoordinateSystem`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['m', 'mm', 'cm', 'n/a']}
	```

EEGGround (metadata)

	name: EEGGround

	description:
	>Description of the location of the ground electrode (for example, `"placed on right mastoid (M2)"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

EEGPlacementScheme (metadata)

	name: EEGPlacementScheme

	description:
	>Placement scheme of EEG electrodes. Either the name of a standardized placement system (for example, `"10-20"`) or a list of standardized electrode names (for example, `["Cz", "Pz"]`).

	schema information:
	```yaml
	{'type': 'string'}
	```

EEGReference (metadata)

	name: EEGReference

	description:
	>General description of the reference scheme used and (when applicable) of location of the reference electrode in the raw recordings (for example, `"left mastoid"`, `"Cz"`, `"CMS"`). If different channels have a different reference, this field should have a general description and the channel specific reference should be defined in the `channels.tsv` file.

	schema information:
	```yaml
	{'type': 'string'}
	```

EMGChannelCount (metadata)

	name: EMGChannelCount

	description:
	>Number of EMG channels.

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

EOGChannelCount (metadata)

	name: EOGChannelCount

	description:
	>Number of EOG channels.

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

EchoTime _sense 1_ (metadata)

	name: EchoTime

	description:
	>The echo time (TE) for the acquisition, specified in seconds. Corresponds to DICOM Tag 0018, 0081 `Echo Time` (please note that the DICOM term is in milliseconds not seconds). The data type number may apply to files from any MRI modality concerned with a single value for this field, or to the files in a [file collection](/99-appendices/10-file-collections.html) where the value of this field is iterated using the [echo entity](/99-appendices/09-entities.html#echo). The data type array provides a value for each volume in a 4D dataset and should only be used when the volume timing is critical for interpretation of the data, such as in [ASL](/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#arterial-spin-labeling-perfusion-data) or variable echo time fMRI sequences.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 's', 'exclusiveMinimum': 0}, {'type': 'array', 'items': {'type': 'number', 'unit': 's', 'exclusiveMinimum': 0}}]}
	```

EchoTime _sense 2_ (metadata)

	name: EchoTime

	description:
	>The time (in seconds) when the echo corresponding to this map was acquired.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's', 'exclusiveMinimum': 0}
	```

EchoTime1 (metadata)

	name: EchoTime1

	description:
	>The time (in seconds) when the first (shorter) echo occurs.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's', 'exclusiveMinimum': 0}
	```

EchoTime2 (metadata)

	name: EchoTime2

	description:
	>The time (in seconds) when the second (longer) echo occurs.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's', 'exclusiveMinimum': 0}
	```

EffectiveEchoSpacing (metadata)

	name: EffectiveEchoSpacing

	description:
	>The "effective" sampling interval, specified in seconds, between lines in the phase-encoding direction, defined based on the size of the reconstructed image in the phase direction. It is frequently, but incorrectly, referred to as "dwell time" (see the `"DwellTime"` parameter for actual dwell time). It is required for unwarping distortions using field maps. Note that beyond just in-plane acceleration, a variety of other manipulations to the phase encoding need to be accounted for properly, including partial fourier, phase oversampling, phase resolution, phase field-of-view and interpolation.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'unit': 's'}
	```

ElectricalStimulation (metadata)

	name: ElectricalStimulation

	description:
	>Boolean field to specify if electrical stimulation was done during the recording (options are `true` or `false`). Parameters for event-like stimulation should be specified in the `events.tsv` file.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

ElectricalStimulationParameters (metadata)

	name: ElectricalStimulationParameters

	description:
	>Free form description of stimulation parameters, such as frequency or shape. Specific onsets can be specified in the events.tsv file. Specific shapes can be described here in freeform text.

	schema information:
	```yaml
	{'type': 'string'}
	```

ElectrodeManufacturer (metadata)

	name: ElectrodeManufacturer

	description:
	>Can be used if all electrodes are of the same manufacturer (for example, `"AD-TECH"`, `"DIXI"`). If electrodes of different manufacturers are used, please use the corresponding table in the `_electrodes.tsv` file.

	schema information:
	```yaml
	{'type': 'string'}
	```

ElectrodeManufacturersModelName (metadata)

	name: ElectrodeManufacturersModelName

	description:
	>If different electrode types are used, please use the corresponding table in the `_electrodes.tsv` file.

	schema information:
	```yaml
	{'type': 'string'}
	```

EpochLength (metadata)

	name: EpochLength

	description:
	>Duration of individual epochs in seconds (for example, `1`) in case of epoched data. If recording was continuous or discontinuous, leave out the field.

	schema information:
	```yaml
	{'type': 'number', 'minimum': 0}
	```

EstimationAlgorithm (metadata)

	name: EstimationAlgorithm

	description:
	>Type of algorithm used to perform fitting (for example, `"linear"`, `"non-linear"`, `"LM"` and such).

	schema information:
	```yaml
	{'type': 'string'}
	```

EstimationReference (metadata)

	name: EstimationReference

	description:
	>Reference to the study/studies on which the implementation is based.

	schema information:
	```yaml
	{'type': 'string'}
	```

EthicsApprovals (metadata)

	name: EthicsApprovals

	description:
	>List of ethics committee approvals of the research protocols and/or protocol identifiers.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'string'}}
	```

FLAIR (suffixes)

	name: Fluid attenuated inversion recovery image

	description:
	>In arbitrary units (arbitrary). Structural images with predominant T2 contribution (also known as T2-FLAIR), in which signal from fluids (for example, CSF) is nulled out by adjusting inversion time, coupled with notably long repetition and echo times.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

FLASH (suffixes)

	name: Fast-Low-Angle-Shot image

	description:
	>FLASH (Fast-Low-Angle-Shot) is a vendor-specific implementation for spoiled gradient echo acquisition. It is commonly used for rapid anatomical imaging and also for many different qMRI applications. When used for a single file, it does not convey any information about the image contrast. When used in a file collection, it may result in conflicts across filenames of different applications. **Change:** Removed from suffixes.

FLUO (suffixes)

	name: Fluorescence microscopy

	description:
	>Fluorescence microscopy imaging data

FiducialsCoordinateSystem (metadata)

	name: FiducialsCoordinateSystem

	description:
	>Defines the coordinate system for the fiducials. Preferably the same as the `"EEGCoordinateSystem"`. See [Appendix VIII](/99-appendices/08-coordinate-systems.html) for a list of restricted keywords for coordinate systems. If `"Other"`, provide definition of the coordinate system in `"FiducialsCoordinateSystemDescription"`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['CTF', 'ElektaNeuromag', '4DBti', 'KitYokogawa', 'ChietiItab', 'Other', 'CapTrak', 'EEGLAB', 'EEGLAB-HJ', 'Other', 'ICBM452AirSpace', 'ICBM452Warp5Space', 'IXI549Space', 'fsaverage', 'fsaverageSym', 'fsLR', 'MNIColin27', 'MNI152Lin', 'MNI152NLin2009aSym', 'MNI152NLin2009bSym', 'MNI152NLin2009cSym', 'MNI152NLin2009aAsym', 'MNI152NLin2009bAsym', 'MNI152NLin2009cAsym', 'MNI152NLin6Sym', 'MNI152NLin6ASym', 'MNI305', 'NIHPD', 'OASIS30AntsOASISAnts', 'OASIS30Atropos', 'Talairach', 'UNCInfant', 'fsaverage3', 'fsaverage4', 'fsaverage5', 'fsaverage6', 'fsaveragesym', 'UNCInfant0V21', 'UNCInfant1V21', 'UNCInfant2V21', 'UNCInfant0V22', 'UNCInfant1V22', 'UNCInfant2V22', 'UNCInfant0V23', 'UNCInfant1V23', 'UNCInfant2V23']}
	```

FiducialsCoordinateSystemDescription (metadata)

	name: FiducialsCoordinateSystemDescription

	description:
	>Free-form text description of the coordinate system. May also include a link to a documentation page or paper describing the system in greater detail.

	schema information:
	```yaml
	{'type': 'string'}
	```

FiducialsCoordinateUnits (metadata)

	name: FiducialsCoordinateUnits

	description:
	>Units in which the coordinates that are  listed in the field `"FiducialsCoordinateSystem"` are represented.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['m', 'mm', 'cm', 'n/a']}
	```

FiducialsCoordinates (metadata)

	name: FiducialsCoordinates

	description:
	>Key:value pairs of the labels and 3-D digitized position of anatomical landmarks, interpreted following the `"FiducialsCoordinateSystem"` (for example, `{"NAS": [12.7,21.3,13.9], "LPA": [5.2,11.3,9.6], "RPA": [20.2,11.3,9.1]}`). Each array MUST contain three numeric values corresponding to x, y, and z axis of the coordinate system in that exact order.

	schema information:
	```yaml
	{'type': 'object', 'additionalProperties': {'type': 'array', 'items': {'type': 'number'}, 'minItems': 3, 'maxItems': 3}}
	```

FiducialsDescription (metadata)

	name: FiducialsDescription

	description:
	>Free-form text description of how the fiducials such as vitamin-E capsules were placed relative to anatomical landmarks, and how the position of the fiducials were measured (for example, `"both with Polhemus and with T1w MRI"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

FlipAngle (metadata)

	name: FlipAngle

	description:
	>Flip angle (FA) for the acquisition, specified in degrees. Corresponds to: DICOM Tag 0018, 1314 `Flip Angle`. The data type number may apply to files from any MRI modality concerned with a single value for this field, or to the files in a [file collection](/99-appendices/10-file-collections.html) where the value of this field is iterated using the [flip entity](/99-appendices/09-entities.html#flip). The data type array provides a value for each volume in a 4D dataset and should only be used when the volume timing is critical for interpretation of the data, such as in [ASL](/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#arterial-spin-labeling-perfusion-data) or variable flip angle fMRI sequences.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 'degree', 'exclusiveMinimum': 0, 'maximum': 360}, {'type': 'array', 'items': {'type': 'number', 'unit': 'degree', 'exclusiveMinimum': 0, 'maximum': 360}}]}
	```

FrameDuration (metadata)

	name: FrameDuration

	description:
	>Time duration of each frame in default unit seconds. This corresponds to DICOM Tag 0018, 1242 `Actual Frame Duration` converted to seconds.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number'}, 'unit': 's'}
	```

FrameTimesStart (metadata)

	name: FrameTimesStart

	description:
	>Start times for all frames relative to `"TimeZero"` in default unit seconds.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number'}, 'unit': 's'}
	```

Funding (metadata)

	name: Funding

	description:
	>List of sources of funding (grant numbers).

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'string'}}
	```

GeneratedBy (metadata)

	name: GeneratedBy

	description:
	>Used to specify provenance of the dataset.

	schema information:
	```yaml
	{'type': 'array', 'minItems': 1, 'items': {'type': 'object', 'properties': {'Name': {'type': 'string'}, 'Version': {'type': 'string'}, 'Description': {'type': 'string'}, 'CodeURL': {'type': 'string', 'format': 'uri'}, 'Container': {'type': 'object', 'properties': {'Type': {'type': 'string'}, 'Tag': {'type': 'string'}, 'URI': {'type': 'string', 'format': 'uri'}}}}}}
	```

GeneticLevel (metadata)

	name: GeneticLevel

	description:
	>Describes the level of analysis. Values MUST be one of `"Genetic"`, `"Genomic"`, `"Epigenomic"`, `"Transcriptomic"`, `"Metabolomic"`, or `"Proteomic"`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string', 'enum': ['Genetic', 'Genomic', 'Epigenomic', 'Transcriptomic', 'Metabolomic', 'Proteomic']}, {'type': 'array', 'items': {'type': 'string', 'enum': ['Genetic', 'Genomic', 'Epigenomic', 'Transcriptomic', 'Metabolomic', 'Proteomic']}}]}
	```

Genetics (metadata)

	name: Genetics

	description:
	>An object containing information about the genetics descriptor.

	schema information:
	```yaml
	{'type': 'object', 'properties': {'Database': {'name': 'Genetics.Database', 'description': '[URI](/02-common-principles.html#uniform-resource-indicator)\nof database where the dataset is hosted.\n', 'type': 'string', 'format': 'uri'}, 'Dataset': {'name': 'Genetics.Dataset', 'description': '[URI](/02-common-principles.html#uniform-resource-indicator)\nwhere data can be retrieved.\n', 'type': 'string', 'format': 'uri'}, 'Descriptors': {'name': 'Genetics.Descriptors', 'description': 'List of relevant descriptors (for example, journal articles) for dataset\nusing a valid\n[URI](/02-common-principles.html#uniform-resource-indicator)\nwhen possible.\n', 'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}}}
	```

Genetics.Database (metadata)

	name: Genetics.Database

	description:
	>[URI](/02-common-principles.html#uniform-resource-indicator) of database where the dataset is hosted.

	schema information:
	```yaml
	{'type': 'string', 'format': 'uri'}
	```

Genetics.Dataset (metadata)

	name: Genetics.Dataset

	description:
	>[URI](/02-common-principles.html#uniform-resource-indicator) where data can be retrieved.

	schema information:
	```yaml
	{'type': 'string', 'format': 'uri'}
	```

Genetics.Descriptors (metadata)

	name: Genetics.Descriptors

	description:
	>List of relevant descriptors (for example, journal articles) for dataset using a valid [URI](/02-common-principles.html#uniform-resource-indicator) when possible.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

GradientSetType (metadata)

	name: GradientSetType

	description:
	>It should be possible to infer the gradient coil from the scanner model. If not, for example because of a custom upgrade or use of a gradient insert set, then the specifications of the actual gradient coil should be reported independently.

	schema information:
	```yaml
	{'type': 'string'}
	```

HED (columns)

	name: HED

	description:
	>Hierarchical Event Descriptor (HED) Tag. See [Appendix III](/99-appendices/03-hed.html) for details.

	schema information:
	```yaml
	{'type': 'string'}
	```

HED (metadata)

	name: HED

	description:
	>Hierarchical Event Descriptor (HED) information, see: [Appendix III](/99-appendices/03-hed.html) for details.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'object', 'additionalProperties': {'type': 'string'}}]}
	```

HEDVersion (metadata)

	name: HEDVersion

	description:
	>If HED tags are used: The version of the HED schema used to validate HED tags for study.

	schema information:
	```yaml
	{'type': 'string'}
	```

Haematocrit (metadata)

	name: Haematocrit

	description:
	>Measured haematocrit, meaning the volume of erythrocytes divided by the volume of whole blood.

	schema information:
	```yaml
	{'type': 'number'}
	```

HardcopyDeviceSoftwareVersion (metadata)

	name: HardcopyDeviceSoftwareVersion

	description:
	>Manufacturer's designation of the software of the device that created this Hardcopy Image (the printer). Corresponds to DICOM Tag 0018, 101A `Hardcopy Device Software Version`.

	schema information:
	```yaml
	{'type': 'string'}
	```

HardwareFilters (metadata)

	name: HardwareFilters

	description:
	>Object of temporal hardware filters applied, or `"n/a"` if the data is not available. Each key:value pair in the JSON object is a name of the filter and an object in which its parameters are defined as key:value pairs. For example, `{"Highpass RC filter": {"Half amplitude cutoff (Hz)": 0.0159, "Roll-off": "6dB/Octave"}}`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'object', 'additionalProperties': {'type': 'object'}}, {'type': 'string', 'enum': ['n/a']}]}
	```

HeadCircumference (metadata)

	name: HeadCircumference

	description:
	>Circumference of the participant's head, expressed in cm (for example, `58`).

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'unit': 'cm'}
	```

HeadCoilCoordinateSystem (metadata)

	name: HeadCoilCoordinateSystem

	description:
	>Defines the coordinate system for the head coils. See [Appendix VIII](/99-appendices/08-coordinate-systems.html) for a list of restricted keywords for coordinate systems. If `"Other"`, provide definition of the coordinate system in `HeadCoilCoordinateSystemDescription`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['CTF', 'ElektaNeuromag', '4DBti', 'KitYokogawa', 'ChietiItab', 'Other', 'CapTrak', 'EEGLAB', 'EEGLAB-HJ', 'Other', 'ICBM452AirSpace', 'ICBM452Warp5Space', 'IXI549Space', 'fsaverage', 'fsaverageSym', 'fsLR', 'MNIColin27', 'MNI152Lin', 'MNI152NLin2009aSym', 'MNI152NLin2009bSym', 'MNI152NLin2009cSym', 'MNI152NLin2009aAsym', 'MNI152NLin2009bAsym', 'MNI152NLin2009cAsym', 'MNI152NLin6Sym', 'MNI152NLin6ASym', 'MNI305', 'NIHPD', 'OASIS30AntsOASISAnts', 'OASIS30Atropos', 'Talairach', 'UNCInfant', 'fsaverage3', 'fsaverage4', 'fsaverage5', 'fsaverage6', 'fsaveragesym', 'UNCInfant0V21', 'UNCInfant1V21', 'UNCInfant2V21', 'UNCInfant0V22', 'UNCInfant1V22', 'UNCInfant2V22', 'UNCInfant0V23', 'UNCInfant1V23', 'UNCInfant2V23']}
	```

HeadCoilCoordinateSystemDescription (metadata)

	name: HeadCoilCoordinateSystemDescription

	description:
	>Free-form text description of the coordinate system. May also include a link to a documentation page or paper describing the system in greater detail.

	schema information:
	```yaml
	{'type': 'string'}
	```

HeadCoilCoordinateUnits (metadata)

	name: HeadCoilCoordinateUnits

	description:
	>Units of the coordinates of `HeadCoilCoordinateSystem`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['m', 'mm', 'cm', 'n/a']}
	```

HeadCoilCoordinates (metadata)

	name: HeadCoilCoordinates

	description:
	>Key:value pairs describing head localization coil labels and their coordinates, interpreted following the `HeadCoilCoordinateSystem` (for example, `{"NAS": [12.7,21.3,13.9], "LPA": [5.2,11.3,9.6], "RPA": [20.2,11.3,9.1]}`). Note that coils are not always placed at locations that have a known anatomical name (for example, for Elekta, Yokogawa systems); in that case generic labels can be used (for example, `{"coil1": [12.2,21.3,12.3], "coil2": [6.7,12.3,8.6], "coil3": [21.9,11.0,8.1]}`). Each array MUST contain three numeric values corresponding to x, y, and z axis of the coordinate system in that exact order.

	schema information:
	```yaml
	{'type': 'object', 'additionalProperties': {'type': 'array', 'items': {'type': 'number'}, 'minItems': 3, 'maxItems': 3}}
	```

HeadCoilFrequency (metadata)

	name: HeadCoilFrequency

	description:
	>List of frequencies (in Hz) used by the head localisation coils ('HLC' in CTF systems, 'HPI' in Elekta, 'COH' in BTi/4D) that track the subject's head position in the MEG helmet (for example, `[293, 307, 314, 321]`).

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 'Hz'}, {'type': 'array', 'items': {'type': 'number', 'unit': 'Hz'}}]}
	```

HowToAcknowledge (metadata)

	name: HowToAcknowledge

	description:
	>Text containing instructions on how researchers using this dataset should acknowledge the original authors. This field can also be used to define a publication that should be cited in publications that use the dataset.

	schema information:
	```yaml
	{'type': 'string'}
	```

IRT1 (suffixes)

	name: Inversion recovery T1 mapping

	description:
	>The IRT1 method involves multiple inversion recovery spin-echo images acquired at different inversion times ([Barral et al. 2010](https://doi.org/10.1002/mrm.22497)).

ImageAcquisitionProtocol (metadata)

	name: ImageAcquisitionProtocol

	description:
	>Description of the image acquisition protocol or [URI](/02-common-principles.html#uniform-resource-indicator) (for example from [protocols.io](https://www.protocols.io/)).

	schema information:
	```yaml
	{'type': 'string'}
	```

ImageDecayCorrected (metadata)

	name: ImageDecayCorrected

	description:
	>Boolean flag specifying whether the image data have been decay-corrected.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

ImageDecayCorrectionTime (metadata)

	name: ImageDecayCorrectionTime

	description:
	>Point in time from which the decay correction was applied with respect to `"TimeZero"` in the default unit seconds.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

Immersion (metadata)

	name: Immersion

	description:
	>Lens immersion medium. If the file format is OME-TIFF, the value MUST be consistent with the `Immersion` OME metadata field.

	schema information:
	```yaml
	{'type': 'string'}
	```

InfusionRadioactivity (metadata)

	name: InfusionRadioactivity

	description:
	>Amount of radioactivity infused into the patient. This value must be less than or equal to the total injected radioactivity (`"InjectedRadioactivity"`). Units should be the same as `"InjectedRadioactivityUnits"`.

	schema information:
	```yaml
	{'type': 'number'}
	```

InfusionSpeed (metadata)

	name: InfusionSpeed

	description:
	>If given, infusion speed.

	schema information:
	```yaml
	{'type': 'number'}
	```

InfusionSpeedUnits (metadata)

	name: InfusionSpeedUnits

	description:
	>Unit of infusion speed (for example, `"mL/s"`).

	schema information:
	```yaml
	{'type': 'string', 'format': 'unit'}
	```

InfusionStart (metadata)

	name: InfusionStart

	description:
	>Time of start of infusion with respect to `"TimeZero"` in the default unit seconds.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

InjectedMass (metadata)

	name: InjectedMass

	description:
	>Total mass of radiolabeled compound injected into subject (for example, `10`). This can be derived as the ratio of the `"InjectedRadioactivity"` and `"MolarRadioactivity"`. **For those tracers in which injected mass is not available (for example FDG) can be set to `"n/a"`)**.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number'}, {'type': 'string', 'enum': ['n/a']}]}
	```

InjectedMassPerWeight (metadata)

	name: InjectedMassPerWeight

	description:
	>Injected mass per kilogram bodyweight.

	schema information:
	```yaml
	{'type': 'number'}
	```

InjectedMassPerWeightUnits (metadata)

	name: InjectedMassPerWeightUnits

	description:
	>Unit format of the injected mass per kilogram bodyweight (for example, `"ug/kg"`).

	schema information:
	```yaml
	{'type': 'string', 'format': 'unit'}
	```

InjectedMassUnits (metadata)

	name: InjectedMassUnits

	description:
	>Unit format of the mass of compound injected (for example, `"ug"` or `"umol"`). **Note this is not required for an FDG acquisition, since it is not available, and SHOULD be set to `"n/a"`**.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string', 'format': 'unit'}, {'type': 'string', 'enum': ['n/a']}]}
	```

InjectedRadioactivity (metadata)

	name: InjectedRadioactivity

	description:
	>Total amount of radioactivity injected into the patient (for example, `400`). For bolus-infusion experiments, this value should be the sum of all injected radioactivity originating from both bolus and infusion. Corresponds to DICOM Tag 0018, 1074 `Radionuclide Total Dose`.

	schema information:
	```yaml
	{'type': 'number'}
	```

InjectedRadioactivityUnits (metadata)

	name: InjectedRadioactivityUnits

	description:
	>Unit format of the specified injected radioactivity (for example, `"MBq"`).

	schema information:
	```yaml
	{'type': 'string', 'format': 'unit'}
	```

InjectedVolume (metadata)

	name: InjectedVolume

	description:
	>Injected volume of the radiotracer in the unit `"mL"`.

	schema information:
	```yaml
	{'type': 'number', 'unit': 'mL'}
	```

InjectionEnd (metadata)

	name: InjectionEnd

	description:
	>Time of end of injection with respect to `"TimeZero"` in the default unit seconds.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

InjectionStart (metadata)

	name: InjectionStart

	description:
	>Time of start of injection with respect to `"TimeZero"` in the default unit seconds. This corresponds to DICOM Tag 0018, 1072 `Contrast/Bolus Start Time` converted to seconds relative to `"TimeZero"`.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

InstitutionAddress (metadata)

	name: InstitutionAddress

	description:
	>The address of the institution in charge of the equipment that produced the measurements.

	schema information:
	```yaml
	{'type': 'string'}
	```

InstitutionName (metadata)

	name: InstitutionName

	description:
	>The name of the institution in charge of the equipment that produced the measurements.

	schema information:
	```yaml
	{'type': 'string'}
	```

InstitutionalDepartmentName (metadata)

	name: InstitutionalDepartmentName

	description:
	>The department in the institution in charge of the equipment that produced the measurements.

	schema information:
	```yaml
	{'type': 'string'}
	```

Instructions (metadata)

	name: Instructions

	description:
	>Text of the instructions given to participants before the recording.

	schema information:
	```yaml
	{'type': 'string'}
	```

IntendedFor (metadata)

	name: IntendedFor

	description:
	>The paths to files for which the associated file is intended to be used. Contains one or more filenames with paths relative to the participant subdirectory. Paths need to use forward slashes instead of backward slashes, regardless of operating system.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string', 'format': 'participant_relative'}, {'type': 'array', 'items': {'type': 'string', 'format': 'participant_relative'}}]}
	```

InversionTime (metadata)

	name: InversionTime

	description:
	>The inversion time (TI) for the acquisition, specified in seconds. Inversion time is the time after the middle of inverting RF pulse to middle of excitation pulse to detect the amount of longitudinal magnetization. Corresponds to DICOM Tag 0018, 0082 `Inversion Time` (please note that the DICOM term is in milliseconds not seconds).

	schema information:
	```yaml
	{'type': 'number', 'unit': 's', 'exclusiveMinimum': 0}
	```

LICENSE (top_level_files)

	name: LICENSE

	description:
	>A `LICENSE` file MAY be provided in addition to the short specification of the used license in the `dataset_description.json` `"License"` field. The `"License"` field and `LICENSE` file MUST correspond. The `LICENSE` file MUST be either in ASCII or UTF-8 encoding.

LabelingDistance (metadata)

	name: LabelingDistance

	description:
	>Distance from the center of the imaging slab to the center of the labeling plane (`(P)CASL`) or the leading edge of the labeling slab (`PASL`), in millimeters. If the labeling is performed inferior to the isocenter, this number should be negative. Based on DICOM macro C.8.13.5.14.

	schema information:
	```yaml
	{'type': 'number', 'unit': 'mm'}
	```

LabelingDuration (metadata)

	name: LabelingDuration

	description:
	>Total duration of the labeling pulse train, in seconds, corresponding to the temporal width of the labeling bolus for `"PCASL"` or `"CASL"`. In case all control-label volumes (or deltam or CBF) have the same `LabelingDuration`, a scalar must be specified. In case the control-label volumes (or deltam or cbf) have a different `"LabelingDuration"`, an array of numbers must be specified, for which any `m0scan` in the timeseries has a `"LabelingDuration"` of zero. In case an array of numbers is provided, its length should be equal to the number of volumes specified in `*_aslcontext.tsv`. Corresponds to DICOM Tag 0018, 9258 `ASL Pulse Train Duration`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'minimum': 0, 'unit': 's'}, {'type': 'array', 'items': {'type': 'number', 'unit': 's', 'minimum': 0}}]}
	```

LabelingEfficiency (metadata)

	name: LabelingEfficiency

	description:
	>Labeling efficiency, specified as a number between zero and one, only if obtained externally (for example phase-contrast based).

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0}
	```

LabelingLocationDescription (metadata)

	name: LabelingLocationDescription

	description:
	>Description of the location of the labeling plane (`"CASL"` or `"PCASL"`) or the labeling slab (`"PASL"`) that cannot be captured by fields `LabelingOrientation` or `LabelingDistance`. May include a link to an anonymized screenshot of the planning of the labeling slab/plane with respect to the imaging slab or slices `*_asllabeling.jpg`. Based on DICOM macro C.8.13.5.14.

	schema information:
	```yaml
	{'type': 'string'}
	```

LabelingOrientation (metadata)

	name: LabelingOrientation

	description:
	>Orientation of the labeling plane (`(P)CASL`) or slab (`PASL`). The direction cosines of a normal vector perpendicular to the ASL labeling slab or plane with respect to the patient. Corresponds to DICOM Tag 0018, 9255 `ASL Slab Orientation`.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number'}}
	```

LabelingPulseAverageB1 (metadata)

	name: LabelingPulseAverageB1

	description:
	>The average B1-field strength of the RF labeling pulses, in microteslas. As an alternative, `"LabelingPulseFlipAngle"` can be provided.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'unit': 'uT'}
	```

LabelingPulseAverageGradient (metadata)

	name: LabelingPulseAverageGradient

	description:
	>The average labeling gradient, in milliteslas per meter.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'unit': 'mT/m'}
	```

LabelingPulseDuration (metadata)

	name: LabelingPulseDuration

	description:
	>Duration of the individual labeling pulses, in milliseconds.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'unit': 'ms'}
	```

LabelingPulseFlipAngle (metadata)

	name: LabelingPulseFlipAngle

	description:
	>The flip angle of a single labeling pulse, in degrees, which can be given as an alternative to `"LabelingPulseAverageB1"`.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'maximum': 360, 'unit': 'degree'}
	```

LabelingPulseInterval (metadata)

	name: LabelingPulseInterval

	description:
	>Delay between the peaks of the individual labeling pulses, in milliseconds.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'unit': 'ms'}
	```

LabelingPulseMaximumGradient (metadata)

	name: LabelingPulseMaximumGradient

	description:
	>The maximum amplitude of the gradient switched on during the application of the labeling RF pulse(s), in milliteslas per meter.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'unit': 'mT/m'}
	```

LabelingSlabThickness (metadata)

	name: LabelingSlabThickness

	description:
	>Thickness of the labeling slab in millimeters. For non-selective FAIR a zero is entered. Corresponds to DICOM Tag 0018, 9254 `ASL Slab Thickness`.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'unit': 'mm'}
	```

Levels (metadata)

	name: Levels

	description:
	>For categorical variables: An object of possible values (keys) and their descriptions (values).

	schema information:
	```yaml
	{'type': 'object', 'additionalProperties': {'type': 'string'}}
	```

License (metadata)

	name: License

	description:
	>The license for the dataset. The use of license name abbreviations is RECOMMENDED for specifying a license (see [Appendix II](/99-appendices/02-licenses.html)). The corresponding full license text MAY be specified in an additional `LICENSE` file.

	schema information:
	```yaml
	{'type': 'string'}
	```

LongName (metadata)

	name: LongName

	description:
	>Long (unabbreviated) name of the column.

	schema information:
	```yaml
	{'type': 'string'}
	```

LookLocker (metadata)

	name: LookLocker

	description:
	>Boolean indicating if a Look-Locker readout is used.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

M0Estimate (metadata)

	name: M0Estimate

	description:
	>A single numerical whole-brain M0 value (referring to the M0 of blood), only if obtained externally (for example retrieved from CSF in a separate measurement).

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0}
	```

M0Type (metadata)

	name: M0Type

	description:
	>Describes the presence of M0 information. `"Separate"` means that a separate `*_m0scan.nii[.gz]` is present. `"Included"` means that an m0scan volume is contained within the current `*_asl.nii[.gz]`. `"Estimate"` means that a single whole-brain M0 value is provided. `"Absent"` means that no specific M0 information is present.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['Separate', 'Included', 'Estimate', 'Absent']}
	```

M0map (suffixes)

	name: Equilibrium magnetization (M0) map

	description:
	>In arbitrary units (arbitrary). A common quantitative MRI (qMRI) fitting variable that represents the amount of magnetization at thermal equilibrium. M0 maps are RECOMMENDED to use this suffix if generated by qMRI applications (for example, variable flip angle T1 mapping).

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

MEGChannelCount (metadata)

	name: MEGChannelCount

	description:
	>Number of MEG channels (for example, `275`).

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

MEGCoordinateSystem (metadata)

	name: MEGCoordinateSystem

	description:
	>Defines the coordinate system for the MEG sensors. See [Appendix VIII](/99-appendices/08-coordinate-systems.html) for a list of restricted keywords for coordinate systems. If `"Other"`, provide definition of the coordinate system in `"MEGCoordinateSystemDescription"`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['CTF', 'ElektaNeuromag', '4DBti', 'KitYokogawa', 'ChietiItab', 'Other', 'CapTrak', 'EEGLAB', 'EEGLAB-HJ', 'Other', 'ICBM452AirSpace', 'ICBM452Warp5Space', 'IXI549Space', 'fsaverage', 'fsaverageSym', 'fsLR', 'MNIColin27', 'MNI152Lin', 'MNI152NLin2009aSym', 'MNI152NLin2009bSym', 'MNI152NLin2009cSym', 'MNI152NLin2009aAsym', 'MNI152NLin2009bAsym', 'MNI152NLin2009cAsym', 'MNI152NLin6Sym', 'MNI152NLin6ASym', 'MNI305', 'NIHPD', 'OASIS30AntsOASISAnts', 'OASIS30Atropos', 'Talairach', 'UNCInfant', 'fsaverage3', 'fsaverage4', 'fsaverage5', 'fsaverage6', 'fsaveragesym', 'UNCInfant0V21', 'UNCInfant1V21', 'UNCInfant2V21', 'UNCInfant0V22', 'UNCInfant1V22', 'UNCInfant2V22', 'UNCInfant0V23', 'UNCInfant1V23', 'UNCInfant2V23']}
	```

MEGCoordinateSystemDescription (metadata)

	name: MEGCoordinateSystemDescription

	description:
	>Free-form text description of the coordinate system. May also include a link to a documentation page or paper describing the system in greater detail.

	schema information:
	```yaml
	{'type': 'string'}
	```

MEGCoordinateUnits (metadata)

	name: MEGCoordinateUnits

	description:
	>Units of the coordinates of `"MEGCoordinateSystem"`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['m', 'mm', 'cm', 'n/a']}
	```

MEGRE (suffixes)

	name: Multi-echo Gradient Recalled Echo

	description:
	>Anatomical gradient echo images acquired at different echo times. Please note that this suffix is not intended for the logical grouping of images acquired using an Echo Planar Imaging (EPI) readout.

MEGREFChannelCount (metadata)

	name: MEGREFChannelCount

	description:
	>Number of MEG reference channels (for example, `23`). For systems without such channels (for example, Neuromag Vectorview), `MEGREFChannelCount` should be set to `0`.

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

MESE (suffixes)

	name: Multi-echo Spin Echo

	description:
	>The MESE method involves multiple spin echo images acquired at different echo times and is primarily used for T2 mapping. Please note that this suffix is not intended for the logical grouping of images acquired using an Echo Planar Imaging (EPI) readout.

MP2RAGE (suffixes)

	name: Magnetization Prepared Two Gradient Echoes

	description:
	>The MP2RAGE method is a special protocol that collects several images at different flip angles and inversion times to create a parametric T1map by combining the magnitude and phase images ([Marques et al. 2010](https://doi.org/10.1016/j.neuroimage.2009.10.002)).

MPE (suffixes)

	name: Multi-photon excitation microscopy

	description:
	>Multi-photon excitation microscopy imaging data

MPM (suffixes)

	name: Multi-parametric Mapping

	description:
	>The MPM approaches (a.k.a hMRI) involves the acquisition of highly-similar anatomical images that differ in terms of application of a magnetization transfer RF pulse (MTon or MToff), flip angle and (optionally) echo time and magnitue/phase parts ([Weiskopf et al. 2013](https://doi.org/10.3389/fnins.2013.00095)). See [here](https://owncloud.gwdg.de/index.php/s/iv2TOQwGy4FGDDZ) for suggested MPM acquisition protocols.

MRAcquisitionType (metadata)

	name: MRAcquisitionType

	description:
	>Type of sequence readout. Corresponds to DICOM Tag 0018, 0023 `MR Acquisition Type`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['2D', '3D']}
	```

MRTransmitCoilSequence (metadata)

	name: MRTransmitCoilSequence

	description:
	>This is a relevant field if a non-standard transmit coil is used. Corresponds to DICOM Tag 0018, 9049 `MR Transmit Coil Sequence`.

	schema information:
	```yaml
	{'type': 'string'}
	```

MTNumberOfPulses (metadata)

	name: MTNumberOfPulses

	description:
	>The number of magnetization transfer RF pulses applied before the readout.

	schema information:
	```yaml
	{'type': 'number'}
	```

MTOffsetFrequency (metadata)

	name: MTOffsetFrequency

	description:
	>The frequency offset of the magnetization transfer pulse with respect to the central H1 Larmor frequency in Hertz (Hz).

	schema information:
	```yaml
	{'type': 'number', 'unit': 'Hz'}
	```

MTPulseBandwidth (metadata)

	name: MTPulseBandwidth

	description:
	>The excitation bandwidth of the magnetization transfer pulse in Hertz (Hz).

	schema information:
	```yaml
	{'type': 'number', 'unit': 'Hz'}
	```

MTPulseDuration (metadata)

	name: MTPulseDuration

	description:
	>Duration of the magnetization transfer RF pulse in seconds.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

MTPulseShape (metadata)

	name: MTPulseShape

	description:
	>Shape of the magnetization transfer RF pulse waveform. The value `"GAUSSHANN"` refers to a Gaussian pulse with a Hanning window. The value `"SINCHANN"` refers to a sinc pulse with a Hanning window. The value `"SINCGAUSS"` refers to a sinc pulse with a Gaussian window.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['HARD', 'GAUSSIAN', 'GAUSSHANN', 'SINC', 'SINCHANN', 'SINCGAUSS', 'FERMI']}
	```

MTR (suffixes)

	name: Magnetization Transfer Ratio

	description:
	>This method is to calculate a semi-quantitative magnetization transfer ratio map.

MTRmap (suffixes)

	name: Magnetization transfer ratio image

	description:
	>In arbitrary units (arbitrary). MTR maps are REQUIRED to use this suffix regardless of the method used to generate them. MTRmap intensity values are RECOMMENDED to be represented in percentage in the range of 0-100%.

	schema information:
	```yaml
	{'unit': 'arbitrary', 'minValue': 0, 'maxValue': 100}
	```

MTS (suffixes)

	name: Magnetization transfer saturation

	description:
	>This method is to calculate a semi-quantitative magnetization transfer saturation index map. The MTS method involves three sets of anatomical images that differ in terms of application of a magnetization transfer RF pulse (MTon or MToff) and flip angle ([Helms et al. 2008](https://doi.org/10.1002/mrm.21732)).

MTState (metadata)

	name: MTState

	description:
	>Boolean stating whether the magnetization transfer pulse is applied. Corresponds to DICOM Tag 0018, 9020 `Magnetization Transfer`.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

MTVmap (suffixes)

	name: Macromolecular tissue volume (MTV) image

	description:
	>In arbitrary units (arbitrary). MTV maps are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

MTsat (suffixes)

	name: Magnetization transfer saturation image

	description:
	>In arbitrary units (arbitrary). MTsat maps are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

MWFmap (suffixes)

	name: Myelin water fraction image

	description:
	>In arbitrary units (arbitrary). MWF maps are REQUIRED to use this suffix regardless of the method used to generate them. MWF intensity values are RECOMMENDED to be represented in percentage in the range of 0-100%.

	schema information:
	```yaml
	{'unit': 'arbitrary', 'minValue': 0, 'maxValue': 100}
	```

MagneticFieldStrength (metadata)

	name: MagneticFieldStrength

	description:
	>Nominal field strength of MR magnet in Tesla. Corresponds to DICOM Tag 0018, 0087 `Magnetic Field Strength`.

	schema information:
	```yaml
	{'type': 'number'}
	```

Magnification (metadata)

	name: Magnification

	description:
	>Lens magnification (for example: `40`). If the file format is OME-TIFF, the value MUST be consistent with the `"NominalMagnification"` OME metadata field.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0}
	```

Manual (metadata)

	name: Manual

	description:
	>Indicates if the segmentation was performed manually or via an automated process.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

Manufacturer (metadata)

	name: Manufacturer

	description:
	>Manufacturer of the equipment that produced the measurements.

	schema information:
	```yaml
	{'type': 'string'}
	```

ManufacturersModelName (metadata)

	name: ManufacturersModelName

	description:
	>Manufacturer's model name of the equipment that produced the measurements.

	schema information:
	```yaml
	{'type': 'string'}
	```

MatrixCoilMode (metadata)

	name: MatrixCoilMode

	description:
	>(If used) A method for reducing the number of independent channels by combining in analog the signals from multiple coil elements. There are typically different default modes when using un-accelerated or accelerated (for example, `"GRAPPA"`, `"SENSE"`) imaging.

	schema information:
	```yaml
	{'type': 'string'}
	```

MaxMovement (metadata)

	name: MaxMovement

	description:
	>Maximum head movement (in mm) detected during the recording, as measured by the head localisation coils (for example, `4.8`).

	schema information:
	```yaml
	{'type': 'number', 'unit': 'mm'}
	```

MeasurementToolMetadata (metadata)

	name: MeasurementToolMetadata

	description:
	>A description of the measurement tool as a whole. Contains two fields: `"Description"` and `"TermURL"`. `"Description"` is a free text description of the measurement tool. `"TermURL"` is a URL to an entity in an ontology corresponding to this tool.

	schema information:
	```yaml
	{'type': 'object', 'properties': {'TermURL': {'type': 'string', 'format': 'uri'}, 'Description': {'type': 'string'}}}
	```

MetaboliteAvail (metadata)

	name: MetaboliteAvail

	description:
	>Boolean that specifies if metabolite measurements are available. If `true`, the `metabolite_parent_fraction` column MUST be present in the corresponding `*_blood.tsv` file.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

MetaboliteMethod (metadata)

	name: MetaboliteMethod

	description:
	>Method used to measure metabolites.

	schema information:
	```yaml
	{'type': 'string'}
	```

MetaboliteRecoveryCorrectionApplied (metadata)

	name: MetaboliteRecoveryCorrectionApplied

	description:
	>Metabolite recovery correction from the HPLC, for tracers where it changes with time postinjection. If `true`, the `hplc_recovery_fractions` column MUST be present in the corresponding `*_blood.tsv` file.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

MiscChannelCount (metadata)

	name: MiscChannelCount

	description:
	>Number of miscellaneous analog channels for auxiliary signals.

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

MixingTime (metadata)

	name: MixingTime

	description:
	>In the context of a stimulated- and spin-echo 3D EPI sequence for B1+ mapping, corresponds to the interval between spin- and stimulated-echo pulses. In the context of a diffusion-weighted double spin-echo sequence, corresponds to the interval between two successive diffusion sensitizing gradients, specified in seconds.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

ModeOfAdministration (metadata)

	name: ModeOfAdministration

	description:
	>Mode of administration of the injection (for example, `"bolus"`, `"infusion"`, or `"bolus-infusion"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

MolarActivity (metadata)

	name: MolarActivity

	description:
	>Molar activity of compound injected. Corresponds to DICOM Tag 0018, 1077 `Radiopharmaceutical Specific Activity`.

	schema information:
	```yaml
	{'type': 'number'}
	```

MolarActivityMeasTime (metadata)

	name: MolarActivityMeasTime

	description:
	>Time to which molar radioactivity measurement above applies in the default unit `"hh:mm:ss"`.

	schema information:
	```yaml
	{'type': 'string', 'format': 'time'}
	```

MolarActivityUnits (metadata)

	name: MolarActivityUnits

	description:
	>Unit of the specified molar radioactivity (for example, `"GBq/umol"`).

	schema information:
	```yaml
	{'type': 'string', 'format': 'unit'}
	```

MultibandAccelerationFactor (metadata)

	name: MultibandAccelerationFactor

	description:
	>The multiband factor, for multiband acquisitions.

	schema information:
	```yaml
	{'type': 'number'}
	```

MultipartID (metadata)

	name: MultipartID

	description:
	>A unique (per participant) label tagging DWI runs that are part of a multipart scan.

	schema information:
	```yaml
	{'type': 'string'}
	```

NLO (suffixes)

	name: Nonlinear optical microscopy

	description:
	>Nonlinear optical microscopy imaging data

Name (metadata)

	name: Name

	description:
	>Name of the dataset.

	schema information:
	```yaml
	{'type': 'string'}
	```

NegativeContrast (metadata)

	name: NegativeContrast

	description:
	>`true` or `false` value specifying whether increasing voxel intensity (within sample voxels) denotes a decreased value with respect to the contrast suffix. This is commonly the case when Cerebral Blood Volume is estimated via usage of a contrast agent in conjunction with a T2\* weighted acquisition protocol.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

None (extensions)

	name: No extension

	description:
	>A file with no extension.

NonlinearGradientCorrection (metadata)

	name: NonlinearGradientCorrection

	description:
	>Boolean stating if the image saved has been corrected for gradient nonlinearities by the scanner sequence.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

NumberOfVolumesDiscardedByScanner (metadata)

	name: NumberOfVolumesDiscardedByScanner

	description:
	>Number of volumes ("dummy scans") discarded by the scanner (as opposed to those discarded by the user post hoc) before saving the imaging file. For example, a sequence that automatically discards the first 4 volumes before saving would have this field as 4. A sequence that does not discard dummy scans would have this set to 0. Please note that the onsets recorded in the `events.tsv` file should always refer to the beginning of the acquisition of the first volume in the corresponding imaging file - independent of the value of `"NumberOfVolumesDiscardedByScanner"` field.

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

NumberOfVolumesDiscardedByUser (metadata)

	name: NumberOfVolumesDiscardedByUser

	description:
	>Number of volumes ("dummy scans") discarded by the user before including the file in the dataset. If possible, including all of the volumes is strongly recommended. Please note that the onsets recorded in the `events.tsv` file should always refer to the beginning of the acquisition of the first volume in the corresponding imaging file - independent of the value of `"NumberOfVolumesDiscardedByUser"` field.

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

NumberShots (metadata)

	name: NumberShots

	description:
	>The number of RF excitations needed to reconstruct a slice or volume (may be referred to as partition). Please mind that this is not the same as Echo Train Length which denotes the number of k-space lines collected after excitation in a multi-echo readout. The data type array is applicable for specifying this parameter before and after the k-space center is sampled. Please see [`"NumberShots"` metadata field](/99-appendices/11-qmri.html#numbershots-metadata-field) in the qMRI appendix for corresponding calculations.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number'}, {'type': 'array', 'items': {'type': 'number'}}]}
	```

NumericalAperture (metadata)

	name: NumericalAperture

	description:
	>Lens numerical aperture (for example: `1.4`). If the file format is OME-TIFF, the value MUST be consistent with the `LensNA` OME metadata field.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0}
	```

OCT (suffixes)

	name: Optical coherence tomography

	description:
	>Optical coherence tomography imaging data

OperatingSystem (metadata)

	name: OperatingSystem

	description:
	>Operating system used to run the stimuli presentation software (for formatting recommendations, see examples below this table).

	schema information:
	```yaml
	{'type': 'string'}
	```

OtherAcquisitionParameters (metadata)

	name: OtherAcquisitionParameters

	description:
	>Description of other relevant image acquisition parameters.

	schema information:
	```yaml
	{'type': 'string'}
	```

PASLType (metadata)

	name: PASLType

	description:
	>Type of the labeling pulse of the `PASL` labeling, for example `"FAIR"`, `"EPISTAR"`, or `"PICORE"`.

	schema information:
	```yaml
	{'type': 'string'}
	```

PC (suffixes)

	name: Phase-contrast microscopy

	description:
	>Phase-contrast microscopy imaging data

PCASLType (metadata)

	name: PCASLType

	description:
	>The type of gradient pulses used in the `control` condition.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['balanced', 'unbalanced']}
	```

PD (suffixes)

	name: Proton density image

	description:
	>Ambiguous, may refer to a parametric image or to a conventional image. **Change:** Replaced by `PDw` or `PDmap`.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

PDT2 (suffixes)

	name: PD and T2 weighted image

	description:
	>In arbitrary units (arbitrary). PDw and T2w images acquired using a dual echo FSE sequence through view sharing process ([Johnson et al. 1994](https://pubmed.ncbi.nlm.nih.gov/8010268/)).

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

PDT2map (suffixes)

	name: Combined PD/T2 image

	description:
	>In arbitrary units (arbitrary). Combined PD/T2 maps are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

PDmap (suffixes)

	name: Proton density image

	description:
	>In arbitrary units (arbitrary). PD maps are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

PDw (suffixes)

	name: Proton density (PD) weighted image

	description:
	>In arbitrary units (arbitrary). The contrast of these images is mainly determined by spatial variations in the spin density (1H) of the imaged specimen. In spin-echo sequences this contrast is achieved at short repetition and long echo times. In a gradient-echo acquisition, PD weighting dominates the contrast at long repetition and short echo times, and at small flip angles.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

PLI (suffixes)

	name: Polarized-light microscopy

	description:
	>Polarized-light microscopy imaging data

ParallelAcquisitionTechnique (metadata)

	name: ParallelAcquisitionTechnique

	description:
	>The type of parallel imaging used (for example `"GRAPPA"`, `"SENSE"`). Corresponds to DICOM Tag 0018, 9078 `Parallel Acquisition Technique`.

	schema information:
	```yaml
	{'type': 'string'}
	```

ParallelReductionFactorInPlane (metadata)

	name: ParallelReductionFactorInPlane

	description:
	>The parallel imaging (for instance, GRAPPA) factor. Use the denominator of the fraction of k-space encoded for each slice. For example, 2 means half of k-space is encoded. Corresponds to DICOM Tag 0018, 9069 `Parallel Reduction Factor In-plane`.

	schema information:
	```yaml
	{'type': 'number'}
	```

PartialFourier (metadata)

	name: PartialFourier

	description:
	>The fraction of partial Fourier information collected. Corresponds to DICOM Tag 0018, 9081 `Partial Fourier`.

	schema information:
	```yaml
	{'type': 'number'}
	```

PartialFourierDirection (metadata)

	name: PartialFourierDirection

	description:
	>The direction where only partial Fourier information was collected. Corresponds to DICOM Tag 0018, 9036 `Partial Fourier Direction`.

	schema information:
	```yaml
	{'type': 'string'}
	```

PharmaceuticalDoseAmount (metadata)

	name: PharmaceuticalDoseAmount

	description:
	>Dose amount of pharmaceutical coadministered with tracer.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number'}, {'type': 'array', 'items': {'type': 'number'}}]}
	```

PharmaceuticalDoseRegimen (metadata)

	name: PharmaceuticalDoseRegimen

	description:
	>Details of the pharmaceutical dose regimen. Either adequate description or short-code relating to regimen documented elsewhere (for example, `"single oral bolus"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

PharmaceuticalDoseTime (metadata)

	name: PharmaceuticalDoseTime

	description:
	>Time of administration of pharmaceutical dose, relative to time zero. For an infusion, this should be a vector with two elements specifying the start and end of the infusion period. For more complex dose regimens, the regimen description should be complete enough to enable unambiguous interpretation of `"PharmaceuticalDoseTime"`. Unit format of the specified pharmaceutical dose time MUST be seconds.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 's'}, {'type': 'array', 'items': {'type': 'number', 'unit': 's'}}]}
	```

PharmaceuticalDoseUnits (metadata)

	name: PharmaceuticalDoseUnits

	description:
	>Unit format relating to pharmaceutical dose (for example, `"mg"` or `"mg/kg"`).

	schema information:
	```yaml
	{'type': 'string', 'format': 'unit'}
	```

PharmaceuticalName (metadata)

	name: PharmaceuticalName

	description:
	>Name of pharmaceutical coadministered with tracer.

	schema information:
	```yaml
	{'type': 'string'}
	```

PhaseEncodingDirection (metadata)

	name: PhaseEncodingDirection

	description:
	>The letters `i`, `j`, `k` correspond to the first, second and third axis of the data in the NIFTI file. The polarity of the phase encoding is assumed to go from zero index to maximum index unless `-` sign is present (then the order is reversed - starting from the highest index instead of zero). `PhaseEncodingDirection` is defined as the direction along which phase is was modulated which may result in visible distortions. Note that this is not the same as the DICOM term `InPlanePhaseEncodingDirection` which can have `ROW` or `COL` values.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['i', 'j', 'k', 'i-', 'j-', 'k-']}
	```

PixelSize (metadata)

	name: PixelSize

	description:
	>A 2- or 3-number array of the physical size of a pixel, either `[PixelSizeX, PixelSizeY]` or `[PixelSizeX, PixelSizeY, PixelSizeZ]`, where X is the width, Y the height and Z the depth. If the file format is OME-TIFF, these values need to be consistent with `PhysicalSizeX`, `PhysicalSizeY` and `PhysicalSizeZ` OME metadata fields, after converting in `PixelSizeUnits` according to `PhysicalSizeXunit`, `PhysicalSizeYunit` and `PhysicalSizeZunit` OME fields.

	schema information:
	```yaml
	{'type': 'array', 'minItems': 2, 'maxItems': 3, 'items': {'type': 'number', 'minimum': 0}}
	```

PixelSizeUnits (metadata)

	name: PixelSizeUnits

	description:
	>Unit format of the specified `"PixelSize"`. MUST be one of: `"mm"` (millimeter), `"um"` (micrometer) or `"nm"` (nanometer).

	schema information:
	```yaml
	{'type': 'string', 'enum': ['mm', 'um', 'nm']}
	```

PlasmaAvail (metadata)

	name: PlasmaAvail

	description:
	>Boolean that specifies if plasma measurements are available.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

PlasmaFreeFraction (metadata)

	name: PlasmaFreeFraction

	description:
	>Measured free fraction in plasma, meaning the concentration of free compound in plasma divided by total concentration of compound in plasma (Units: 0-100%).

	schema information:
	```yaml
	{'type': 'number', 'minimum': 0, 'maximum': 100}
	```

PlasmaFreeFractionMethod (metadata)

	name: PlasmaFreeFractionMethod

	description:
	>Method used to estimate free fraction.

	schema information:
	```yaml
	{'type': 'string'}
	```

PostLabelingDelay (metadata)

	name: PostLabelingDelay

	description:
	>This is the postlabeling delay (PLD) time, in seconds, after the end of the labeling (for `"CASL"` or `"PCASL"`) or middle of the labeling pulse (for `"PASL"`) until the middle of the excitation pulse applied to the imaging slab (for 3D acquisition) or first slice (for 2D acquisition). Can be a number (for a single-PLD time series) or an array of numbers (for multi-PLD and Look-Locker). In the latter case, the array of numbers contains the PLD of each volume, namely each `control` and `label`, in the acquisition order. Any image within the time-series without a PLD, for example an `m0scan`, is indicated by a zero. Based on DICOM Tags 0018, 9079 `Inversion Times` and 0018, 0082 `InversionTime`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'exclusiveMinimum': 0, 'unit': 's'}, {'type': 'array', 'items': {'type': 'number', 'exclusiveMinimum': 0, 'unit': 's'}}]}
	```

PowerLineFrequency (metadata)

	name: PowerLineFrequency

	description:
	>Frequency (in Hz) of the power grid at the geographical location of the instrument (for example, `50` or `60`).

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'exclusiveMinimum': 0, 'unit': 'Hz'}, {'type': 'string', 'enum': ['n/a']}]}
	```

PromptRate (metadata)

	name: PromptRate

	description:
	>Prompt rate for each frame (same units as `Units`, for example, `"Bq/mL"`).

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number'}}
	```

PulseSequenceDetails (metadata)

	name: PulseSequenceDetails

	description:
	>Information beyond pulse sequence type that identifies the specific pulse sequence used (for example, `"Standard Siemens Sequence distributed with the VB17 software"`, `"Siemens WIP ### version #.##,"` or `"Sequence written by X using a version compiled on MM/DD/YYYY"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

PulseSequenceType (metadata)

	name: PulseSequenceType

	description:
	>A general description of the pulse sequence used for the scan (for example, `"MPRAGE"`, `"Gradient Echo EPI"`, `"Spin Echo EPI"`, `"Multiband gradient echo EPI"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

Purity (metadata)

	name: Purity

	description:
	>Purity of the radiolabeled compound (between 0 and 100%).

	schema information:
	```yaml
	{'type': 'number', 'minimum': 0, 'maximum': 100}
	```

R1map (suffixes)

	name: Longitudinal relaxation rate image

	description:
	>In seconds<sup>-1</sup> (1/s). R1 maps (R1 = 1/T1) are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': '1/s'}
	```

R2map (suffixes)

	name: True transverse relaxation rate image

	description:
	>In seconds<sup>-1</sup> (1/s). R2 maps (R2 = 1/T2) are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': '1/s'}
	```

R2starmap (suffixes)

	name: Observed transverse relaxation rate image

	description:
	>In seconds<sup>-1</sup> (1/s). R2-star maps (R2star = 1/T2star) are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': '1/s'}
	```

RB1COR (suffixes)

	name: RB1COR

	description:
	>Low resolution images acquired by the body coil (in the gantry of the scanner) and the head coil using identical acquisition parameters to generate a combined sensitivity map as described in [Papp et al. (2016)](https://doi.org/10.1002/mrm.26058).

RB1map (suffixes)

	name: RF receive sensitivity map

	description:
	>In arbitrary units (arbitrary). Radio frequency (RF) receive (B1-) sensitivity maps are REQUIRED to use this suffix regardless of the method used to generate them. RB1map intensity values are RECOMMENDED to be represented as percent multiplicative factors such that Amplitude<sub>effective</sub> = B1-<sub>intensity</sub>\*Amplitude<sub>ideal</sub>.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

README (top_level_files)

	name: README

	description:
	>A REQUIRED text file, `README`, SHOULD describe the dataset in more detail. The `README` file MUST be either in ASCII or UTF-8 encoding and MAY have one of the extensions: `.md` ([Markdown](https://www.markdownguide.org/)), `.rst` ([reStructuredText](https://docutils.sourceforge.io/rst.html)), or `.txt`. A BIDS dataset MUST NOT contain more than one `README` file (with or without extension) at its root directory. BIDS does not make any recommendations with regards to the [Markdown flavor](https://www.markdownguide.org/extended-syntax/#lightweight-markup-languages) and does not validate the syntax of Markdown and reStructuredText. The `README` file SHOULD be structured such that its contents can be easily understood even if the used format is not rendered. A guideline for creating a good `README` file can be found in the [bids-starter-kit](https://github.com/bids-standard/bids-starter-kit/blob/master/templates/README).

RandomRate (metadata)

	name: RandomRate

	description:
	>Random rate for each frame (same units as `"Units"`, for example, `"Bq/mL"`).

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number'}}
	```

RawSources (metadata)

	name: RawSources

	description:
	>A list of paths relative to dataset root pointing to the BIDS-Raw file(s) that were used in the creation of this derivative.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'string', 'format': 'dataset_relative'}}
	```

ReceiveCoilActiveElements (metadata)

	name: ReceiveCoilActiveElements

	description:
	>Information describing the active/selected elements of the receiver coil. This does not correspond to a tag in the DICOM ontology. The vendor-defined terminology for active coil elements can go in this field.

	schema information:
	```yaml
	{'type': 'string'}
	```

ReceiveCoilName (metadata)

	name: ReceiveCoilName

	description:
	>Information describing the receiver coil. Corresponds to DICOM Tag 0018, 1250 `Receive Coil Name`, although not all vendors populate that DICOM Tag, in which case this field can be derived from an appropriate private DICOM field.

	schema information:
	```yaml
	{'type': 'string'}
	```

ReconFilterSize (metadata)

	name: ReconFilterSize

	description:
	>Kernel size of post-recon filter (FWHM) in default units `"mm"`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 'mm'}, {'type': 'array', 'items': {'type': 'number', 'unit': 'mm'}}]}
	```

ReconFilterType (metadata)

	name: ReconFilterType

	description:
	>Type of post-recon smoothing (for example, `["Shepp"]`).

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

ReconMethodImplementationVersion (metadata)

	name: ReconMethodImplementationVersion

	description:
	>Identification for the software used, such as name and version.

	schema information:
	```yaml
	{'type': 'string'}
	```

ReconMethodName (metadata)

	name: ReconMethodName

	description:
	>Reconstruction method or algorithm (for example, `"3d-op-osem"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

ReconMethodParameterLabels (metadata)

	name: ReconMethodParameterLabels

	description:
	>Names of reconstruction parameters (for example, `["subsets", "iterations"]`).

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'string'}}
	```

ReconMethodParameterUnits (metadata)

	name: ReconMethodParameterUnits

	description:
	>Unit of reconstruction parameters (for example, `["none", "none"]`).

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'string', 'format': 'unit'}}
	```

ReconMethodParameterValues (metadata)

	name: ReconMethodParameterValues

	description:
	>Values of reconstruction parameters (for example, `[21, 3]`).

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number'}}
	```

RecordingDuration (metadata)

	name: RecordingDuration

	description:
	>Length of the recording in seconds (for example, `3600`).

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

RecordingType (metadata)

	name: RecordingType

	description:
	>Defines whether the recording is `"continuous"`, `"discontinuous"`, or `"epoched"`, where `"epoched"` is limited to time windows about events of interest (for example, stimulus presentations or subject responses).

	schema information:
	```yaml
	{'type': 'string', 'enum': ['continuous', 'epoched', 'discontinuous']}
	```

ReferencesAndLinks (metadata)

	name: ReferencesAndLinks

	description:
	>List of references to publications that contain information on the dataset. A reference may be textual or a [URI](/02-common-principles.html#uniform-resource-indicator).

	schema information:
	```yaml
	{'items': {'type': 'string'}, 'type': 'array'}
	```

RepetitionTime (metadata)

	name: RepetitionTime

	description:
	>The time in seconds between the beginning of an acquisition of one volume and the beginning of acquisition of the volume following it (TR). When used in the context of functional acquisitions this parameter best corresponds to [DICOM Tag 0020, 0110](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0020,0110)): the "time delta between images in a dynamic of functional set of images" but may also be found in [DICOM Tag 0018, 0080](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,0080)): "the period of time in msec between the beginning of a pulse sequence and the beginning of the succeeding (essentially identical) pulse sequence". This definition includes time between scans (when no data has been acquired) in case of sparse acquisition schemes. This value MUST be consistent with the 'pixdim[4]' field (after accounting for units stored in 'xyzt_units' field) in the NIfTI header. This field is mutually exclusive with VolumeTiming.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0, 'unit': 's'}
	```

RepetitionTimeExcitation (metadata)

	name: RepetitionTimeExcitation

	description:
	>The interval, in seconds, between two successive excitations. [DICOM Tag 0018, 0080](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,0080) best refers to this parameter. This field may be used together with the `"RepetitionTimePreparation"` for certain use cases, such as [MP2RAGE](https://doi.org/10.1016/j.neuroimage.2009.10.002). Use `RepetitionTimeExcitation` (in combination with `"RepetitionTimePreparation"` if needed) for anatomy imaging data rather than `"RepetitionTime"` as it is already defined as the amount of time that it takes to acquire a single volume in the [task imaging data](/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#task-including-resting-state-imaging-data) section.

	schema information:
	```yaml
	{'type': 'number', 'minimum': 0, 'unit': 's'}
	```

RepetitionTimePreparation (metadata)

	name: RepetitionTimePreparation

	description:
	>The interval, in seconds, that it takes a preparation pulse block to re-appear at the beginning of the succeeding (essentially identical) pulse sequence block. The data type number may apply to files from any MRI modality concerned with a single value for this field. The data type array provides a value for each volume in a 4D dataset and should only be used when the volume timing is critical for interpretation of the data, such as in [ASL](/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#arterial-spin-labeling-perfusion-data).

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'minimum': 0, 'unit': 's'}, {'type': 'array', 'items': {'type': 'number', 'minimum': 0, 'unit': 's'}}]}
	```

Resolution (metadata)

	name: Resolution

	description:
	>Specifies the interpretation of the resolution keyword. If an object is used, then the keys should be values for the ``res`` entity and values should be descriptions of those ``res`` values.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'object', 'additionalProperties': {'type': 'string'}}]}
	```

S0map (suffixes)

	name: Observed signal amplitude (S0) image

	description:
	>In arbitrary units (arbitrary). For a multi-echo (typically fMRI) sequence, S0 maps index the baseline signal before exponential (T2-star) signal decay. In other words: the exponential of the intercept for a linear decay model across log-transformed echos. For more information, please see, for example, [the tedana documentation](https://tedana.readthedocs.io/en/latest/approach.html#monoexponential-decay-model-fit). S0 maps are RECOMMENDED to use this suffix if derived from an ME-FMRI dataset.

SEEGChannelCount (metadata)

	name: SEEGChannelCount

	description:
	>Number of SEEG channels.

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

SEM (suffixes)

	name: Scanning electron microscopy

	description:
	>Scanning electron microscopy imaging data

SPIM (suffixes)

	name: Selective plane illumination microscopy

	description:
	>Selective plane illumination microscopy imaging data

SR (suffixes)

	name: Super-resolution microscopy

	description:
	>Super-resolution microscopy imaging data

SampleEmbedding (metadata)

	name: SampleEmbedding

	description:
	>Description of the tissue sample embedding (for example: `"Epoxy resin"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

SampleEnvironment (metadata)

	name: SampleEnvironment

	description:
	>Environment in which the sample was imaged. MUST be one of: `"in vivo"`, `"ex vivo"` or `"in vitro"`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['in vivo', 'ex vivo', 'in vitro']}
	```

SampleExtractionInstitution (metadata)

	name: SampleExtractionInstitution

	description:
	>The name of the institution in charge of the extraction of the sample, if different from the institution in charge of the equipment that produced the image.

	schema information:
	```yaml
	{'type': 'string'}
	```

SampleExtractionProtocol (metadata)

	name: SampleExtractionProtocol

	description:
	>Description of the sample extraction protocol or [URI](/02-common-principles.html#uniform-resource-indicator) (for example from [protocols.io](https://www.protocols.io/)).

	schema information:
	```yaml
	{'type': 'string'}
	```

SampleFixation (metadata)

	name: SampleFixation

	description:
	>Description of the tissue sample fixation (for example: `"4% paraformaldehyde, 2% glutaraldehyde"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

SampleOrigin (metadata)

	name: SampleOrigin

	description:
	>Describes from which tissue the genetic information was extracted.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['blood', 'saliva', 'brain', 'csf', 'breast milk', 'bile', 'amniotic fluid', 'other biospecimen']}
	```

SamplePrimaryAntibody (metadata)

	name: SamplePrimaryAntibody

	description:
	>Description(s) of the primary antibody used for immunostaining. Either an [RRID](https://scicrunch.org/resources) or the name, supplier and catalogue number of a commercial antibody. For non-commercial antibodies either an [RRID](https://scicrunch.org/resources) or the host-animal and immunogen used (for examples: `"RRID:AB_2122563"` or `"Rabbit anti-Human HTR5A Polyclonal Antibody, Invitrogen, Catalog # PA1-2453"`). MAY be an array of strings if different antibodies are used in each channel of the file.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

SampleSecondaryAntibody (metadata)

	name: SampleSecondaryAntibody

	description:
	>Description(s) of the secondary antibody used for immunostaining. Either an [RRID](https://scicrunch.org/resources) or the name, supplier and catalogue number of a commercial antibody. For non-commercial antibodies either an [RRID](https://scicrunch.org/resources) or the host-animal and immunogen used (for examples: `"RRID:AB_228322"` or `"Goat anti-Mouse IgM Secondary Antibody, Invitrogen, Catalog # 31172"`). MAY be an array of strings if different antibodies are used in each channel of the file.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

SampleStaining (metadata)

	name: SampleStaining

	description:
	>Description(s) of the tissue sample staining (for example: `"Osmium"`). MAY be an array of strings if different stains are used in each channel of the file (for example: `["LFB", "PLP"]`).

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

SamplingFrequency (metadata)

	name: SamplingFrequency

	description:
	>Sampling frequency (in Hz) of all the data in the recording, regardless of their type (for example, `2400`).

	schema information:
	```yaml
	{'type': 'number', 'unit': 'Hz'}
	```

ScaleFactor (metadata)

	name: ScaleFactor

	description:
	>Scale factor for each frame. This field MUST be defined if the imaging data (`.nii[.gz]`) are scaled. If this field is not defined, then it is assumed that the scaling factor is 1. Defining this field when the scaling factor is 1 is RECOMMENDED, for the sake of clarity.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number'}}
	```

ScanDate (metadata)

	name: ScanDate

	description:
	>Date of scan in the format `"YYYY-MM-DD[Z]"`. This field is DEPRECATED, and this metadata SHOULD be recorded in the `acq_time` column of the corresponding [Scans file](/03-modality-agnostic-files.html#scans-file).

	schema information:
	```yaml
	{'type': 'string', 'format': 'date'}
	```

ScanOptions (metadata)

	name: ScanOptions

	description:
	>Parameters of ScanningSequence. Corresponds to DICOM Tag 0018, 0022 `Scan Options`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

ScanStart (metadata)

	name: ScanStart

	description:
	>Time of start of scan with respect to `TimeZero` in the default unit seconds.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

ScanningSequence (metadata)

	name: ScanningSequence

	description:
	>Description of the type of data acquired. Corresponds to DICOM Tag 0018, 0020 `Scanning Sequence`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

ScatterFraction (metadata)

	name: ScatterFraction

	description:
	>Scatter fraction for each frame (Units: 0-100%).

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number', 'minimum': 0, 'maximum': 100}}
	```

SequenceName (metadata)

	name: SequenceName

	description:
	>Manufacturer's designation of the sequence name. Corresponds to DICOM Tag 0018, 0024 `Sequence Name`.

	schema information:
	```yaml
	{'type': 'string'}
	```

SequenceVariant (metadata)

	name: SequenceVariant

	description:
	>Variant of the ScanningSequence. Corresponds to DICOM Tag 0018, 0021 `Sequence Variant`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'array', 'items': {'type': 'string'}}]}
	```

SinglesRate (metadata)

	name: SinglesRate

	description:
	>Singles rate for each frame (same units as `Units`, for example, `"Bq/mL"`).

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number'}}
	```

SkullStripped (metadata)

	name: SkullStripped

	description:
	>Whether the volume was skull stripped (non-brain voxels set to zero) or not.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

SliceEncodingDirection (metadata)

	name: SliceEncodingDirection

	description:
	>The axis of the NIfTI data along which slices were acquired, and the direction in which `"SliceTiming"` is defined with respect to. `i`, `j`, `k` identifiers correspond to the first, second and third axis of the data in the NIfTI file. A `-` sign indicates that the contents of `"SliceTiming"` are defined in reverse order - that is, the first entry corresponds to the slice with the largest index, and the final entry corresponds to slice index zero. When present, the axis defined by `"SliceEncodingDirection"` needs to be consistent with the `slice_dim` field in the NIfTI header. When absent, the entries in `"SliceTiming"` must be in the order of increasing slice index as defined by the NIfTI header.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['i', 'j', 'k', 'i-', 'j-', 'k-']}
	```

SliceThickness (metadata)

	name: SliceThickness

	description:
	>Slice thickness of the tissue sample in the unit micrometers (`"um"`) (for example: `5`).

	schema information:
	```yaml
	{'type': 'number', 'unit': 'um', 'exclusiveMinimum': 0}
	```

SliceTiming (metadata)

	name: SliceTiming

	description:
	>The time at which each slice was acquired within each volume (frame) of the acquisition. Slice timing is not slice order -- rather, it is a list of times containing the time (in seconds) of each slice acquisition in relation to the beginning of volume acquisition. The list goes through the slices along the slice axis in the slice encoding dimension (see below). Note that to ensure the proper interpretation of the `"SliceTiming"` field, it is important to check if the OPTIONAL `SliceEncodingDirection` exists. In particular, if `"SliceEncodingDirection"` is negative, the entries in `"SliceTiming"` are defined in reverse order with respect to the slice axis, such that the final entry in the `"SliceTiming"` list is the time of acquisition of slice 0. Without this parameter slice time correction will not be possible.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'number', 'minimum': 0, 'unit': 's'}}
	```

SoftwareFilters (metadata)

	name: SoftwareFilters

	description:
	>[Object](https://www.json.org/json-en.html) of temporal software filters applied, or `"n/a"` if the data is not available. Each key:value pair in the JSON object is a name of the filter and an object in which its parameters are defined as key:value pairs (for example, `{"Anti-aliasing filter": {"half-amplitude cutoff (Hz)": 500, "Roll-off": "6dB/Octave"}}`).

	schema information:
	```yaml
	{'anyOf': [{'type': 'object', 'additionalProperties': {'type': 'object'}}, {'type': 'string', 'enum': ['n/a']}]}
	```

SoftwareName (metadata)

	name: SoftwareName

	description:
	>Name of the software that was used to present the stimuli.

	schema information:
	```yaml
	{'type': 'string'}
	```

SoftwareRRID (metadata)

	name: SoftwareRRID

	description:
	>[Research Resource Identifier](https://scicrunch.org/resources) of the software that was used to present the stimuli. Examples: The RRID for Psychtoolbox is 'SCR_002881', and that of PsychoPy is 'SCR_006571'.

	schema information:
	```yaml
	{'type': 'string', 'format': 'rrid'}
	```

SoftwareVersion (metadata)

	name: SoftwareVersion

	description:
	>Version of the software that was used to present the stimuli.

	schema information:
	```yaml
	{'type': 'string'}
	```

SoftwareVersions (metadata)

	name: SoftwareVersions

	description:
	>Manufacturer's designation of software version of the equipment that produced the measurements.

	schema information:
	```yaml
	{'type': 'string'}
	```

SourceDatasets (metadata)

	name: SourceDatasets

	description:
	>Used to specify the locations and relevant attributes of all source datasets. Valid keys in each object include `"URL"`, `"DOI"` (see [URI](/02-common-principles.html#uniform-resource-indicator)), and `"Version"` with [string](https://www.w3schools.com/js/js_json_datatypes.asp) values.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'object', 'properties': {'URL': {'type': 'string', 'format': 'uri'}, 'DOI': {'type': 'string'}, 'Version': {'type': 'string'}}}}
	```

Sources (metadata)

	name: Sources

	description:
	>A list of files with the paths specified relative to dataset root; these files were directly used in the creation of this derivative data file. For example, if a derivative A is used in the creation of another derivative B, which is in turn used to generate C in a chain of A->B->C, C should only list B in `"Sources"`, and B should only list A in `"Sources"`. However, in case both X and Y are directly used in the creation of Z, then Z should list X and Y in `"Sources"`, regardless of whether X was used to generate Y.

	schema information:
	```yaml
	{'type': 'array', 'items': {'type': 'string', 'format': 'dataset_relative'}}
	```

SpatialReference (metadata)

	name: SpatialReference

	description:
	>For images with a single reference, the value MUST be a single string. For images with multiple references, such as surface and volume references, a JSON object MUST be used.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string', 'enum': ['orig']}, {'type': 'string', 'format': 'uri'}, {'type': 'string', 'format': 'dataset_relative'}, {'type': 'object', 'additionalProperties': {'anyOf': [{'type': 'string', 'enum': ['orig']}, {'type': 'string', 'format': 'uri'}, {'type': 'string', 'format': 'dataset_relative'}]}}]}
	```

SpecificRadioactivity (metadata)

	name: SpecificRadioactivity

	description:
	>Specific activity of compound injected. **Note this is not required for an FDG acquisition, since it is not available, and SHOULD be set to `"n/a"`**.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number'}, {'type': 'string', 'enum': ['n/a']}]}
	```

SpecificRadioactivityMeasTime (metadata)

	name: SpecificRadioactivityMeasTime

	description:
	>Time to which specific radioactivity measurement above applies in the default unit `"hh:mm:ss"`.

	schema information:
	```yaml
	{'type': 'string', 'format': 'time'}
	```

SpecificRadioactivityUnits (metadata)

	name: SpecificRadioactivityUnits

	description:
	>Unit format of specified specific radioactivity (for example, `"Bq/g"`). **Note this is not required for an FDG acquisition, since it is not available, and SHOULD be set to `"n/a"`**.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string', 'format': 'unit'}, {'type': 'string', 'enum': ['n/a']}]}
	```

SpoilingGradientDuration (metadata)

	name: SpoilingGradientDuration

	description:
	>The duration of the spoiler gradient lobe in seconds. The duration of a trapezoidal lobe is defined as the summation of ramp-up and plateau times.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

SpoilingGradientMoment (metadata)

	name: SpoilingGradientMoment

	description:
	>Zeroth moment of the spoiler gradient lobe in millitesla times second per meter (mT.s/m).

	schema information:
	```yaml
	{'type': 'number', 'unit': 'mT.s/m'}
	```

SpoilingRFPhaseIncrement (metadata)

	name: SpoilingRFPhaseIncrement

	description:
	>The amount of incrementation described in degrees, which is applied to the phase of the excitation pulse at each TR period for achieving RF spoiling.

	schema information:
	```yaml
	{'type': 'number', 'unit': 'degrees'}
	```

SpoilingState (metadata)

	name: SpoilingState

	description:
	>Boolean stating whether the pulse sequence uses any type of spoiling strategy to suppress residual transverse magnetization.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

SpoilingType (metadata)

	name: SpoilingType

	description:
	>Specifies which spoiling method(s) are used by a spoiled sequence.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['RF', 'GRADIENT', 'COMBINED']}
	```

StartTime (metadata)

	name: StartTime

	description:
	>Start time in seconds in relation to the start of acquisition of the first data sample in the corresponding neural dataset (negative values are allowed).

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

StationName (metadata)

	name: StationName

	description:
	>Institution defined name of the machine that produced the measurements.

	schema information:
	```yaml
	{'type': 'string'}
	```

StimulusPresentation (metadata)

	name: StimulusPresentation

	description:
	>Object containing key value pairs related to the software used to present the stimuli during the experiment, specifically: `"OperatingSystem"`, `"SoftwareName"`, `"SoftwareRRID"`, `"SoftwareVersion"` and `"Code"`. See table below for more information.

	schema information:
	```yaml
	{'type': 'object', 'properties': {'OperatingSystem': {'name': 'OperatingSystem', 'description': 'Operating system used to run the stimuli presentation software\n(for formatting recommendations, see examples below this table).\n', 'type': 'string'}, 'SoftwareName': {'name': 'SoftwareName', 'description': 'Name of the software that was used to present the stimuli.\n', 'type': 'string'}, 'SoftwareRRID': {'name': 'SoftwareRRID', 'description': "[Research Resource Identifier](https://scicrunch.org/resources) of the\nsoftware that was used to present the stimuli.\nExamples: The RRID for Psychtoolbox is 'SCR_002881',\nand that of PsychoPy is 'SCR_006571'.\n", 'type': 'string', 'format': 'rrid'}, 'SoftwareVersion': {'name': 'SoftwareVersion', 'description': 'Version of the software that was used to present the stimuli.\n', 'type': 'string'}, 'Code': {'name': 'Code', 'description': '[URI](/02-common-principles.html#uniform-resource-indicator)\nof the code used to present the stimuli.\nPersistent identifiers such as DOIs are preferred.\nIf multiple versions of code may be hosted at the same location,\nrevision-specific URIs are recommended.\n', 'type': 'string', 'format': 'uri'}}}
	```

SubjectArtefactDescription (metadata)

	name: SubjectArtefactDescription

	description:
	>Freeform description of the observed subject artefact and its possible cause (for example, `"Vagus Nerve Stimulator"`, `"non-removable implant"`). If this field is set to `"n/a"`, it will be interpreted as absence of major source of artifacts except cardiac and blinks.

	schema information:
	```yaml
	{'type': 'string'}
	```

T1map (suffixes)

	name: Longitudinal relaxation time image

	description:
	>In seconds (s). T1 maps are REQUIRED to use this suffix regardless of the method used to generate them. See [this interactive book on T1 mapping](https://qmrlab.org/t1_book/intro) for further reading on T1-mapping.

	schema information:
	```yaml
	{'unit': 's'}
	```

T1rho (suffixes)

	name: T1 in rotating frame (T1 rho) image

	description:
	>In seconds (s). T1-rho maps are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': 's'}
	```

T1w (suffixes)

	name: T1-weighted image

	description:
	>In arbitrary units (arbitrary). The contrast of these images is mainly determined by spatial variations in the longitudinal relaxation time of the imaged specimen. In spin-echo sequences this contrast is achieved at relatively short repetition and echo times. To achieve this weighting in gradient-echo images, again, short repetition and echo times are selected; however, at relatively large flip angles. Another common approach to increase T1 weighting in gradient-echo images is to add an inversion preparation block to the beginning of the imaging sequence (for example, `TurboFLASH` or `MP-RAGE`).

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

T2map (suffixes)

	name: True transverse relaxation time image

	description:
	>In seconds (s). T2 maps are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': 's'}
	```

T2star (suffixes)

	name: T2\* image

	description:
	>Ambiguous, may refer to a parametric image or to a conventional image. **Change:** Replaced by `T2starw` or `T2starmap`.

	schema information:
	```yaml
	{'anyOf': [{'unit': 'arbitrary'}, {'unit': 's'}]}
	```

T2starmap (suffixes)

	name: Observed transverse relaxation time image

	description:
	>In seconds (s). T2-star maps are REQUIRED to use this suffix regardless of the method used to generate them.

	schema information:
	```yaml
	{'unit': 's'}
	```

T2starw (suffixes)

	name: T2star weighted image

	description:
	>In arbitrary units (arbitrary). The contrast of these images is mainly determined by spatial variations in the (observed) transverse relaxation time of the imaged specimen. In spin-echo sequences, this effect is negated as the excitation is followed by an inversion pulse. The contrast of gradient-echo images natively depends on T2-star effects. However, for T2-star variation to dominate the image contrast, gradient-echo acquisitions are carried out at long repetition and echo times, and at small flip angles.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

T2w (suffixes)

	name: T2-weighted image

	description:
	>In arbitrary units (arbitrary). The contrast of these images is mainly determined by spatial variations in the (true) transverse relaxation time of the imaged specimen. In spin-echo sequences this contrast is achieved at relatively long repetition and echo times. Generally, gradient echo sequences are not the most suitable option for achieving T2 weighting, as their contrast natively depends on T2-star rather than on T2.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

TB1AFI (suffixes)

	name: TB1AFI

	description:
	>This method ([Yarnykh 2007](https://doi.org/10.1002/mrm.21120)) calculates a B1<sup>+</sup> map from two images acquired at interleaved (two) TRs with identical RF pulses using a steady-state sequence.

TB1DAM (suffixes)

	name: TB1DAM

	description:
	>The double-angle B1<sup>+</sup> method ([Insko and Bolinger 1993](https://doi.org/10.1006/jmra.1993.1133)) is based on the calculation of the actual angles from signal ratios, collected by two acquisitions at different nominal excitation flip angles. Common sequence types for this application include spin echo and echo planar imaging.

TB1EPI (suffixes)

	name: TB1EPI

	description:
	>This B1<sup>+</sup> mapping method ([Jiru and Klose 2006](https://doi.org/10.1002/mrm.21083)) is based on two EPI readouts to acquire spin echo (SE) and stimulated echo (STE) images at multiple flip angles in one sequence, used in the calculation of deviations from the nominal flip angle.

TB1RFM (suffixes)

	name: TB1RFM

	description:
	>The result of a Siemens `rf_map` product sequence. This sequence produces two images. The first image appears like an anatomical image and the second output is a scaled flip angle map.

TB1SRGE (suffixes)

	name: TB1SRGE

	description:
	>Saturation-prepared with 2 rapid gradient echoes (SA2RAGE) uses a ratio of two saturation recovery images with different time delays, and a simulated look-up table to estimate B1+ ([Eggenschwiler et al. 2011](https://doi.org/10.1002/mrm.23145)). This sequence can also be used in conjunction with MP2RAGE T1 mapping to iteratively improve B1+ and T1 map estimation ([Marques & Gruetter 2013](https://doi.org/10.1371/journal.pone.0069294)).

TB1TFL (suffixes)

	name: TB1TFL

	description:
	>The result of a Siemens `tfl_b1_map` product sequence. This sequence produces two images. The first image appears like an anatomical image and the second output is a scaled flip angle map.

TB1map (suffixes)

	name: RF transmit field image

	description:
	>In arbitrary units (arbitrary). Radio frequency (RF) transmit (B1+) field maps are REQUIRED to use this suffix regardless of the method used to generate them. TB1map intensity values are RECOMMENDED to be represented as percent multiplicative factors such that FlipAngle<sub>effective</sub> = B1+<sub>intensity</sub>\*FlipAngle<sub>nominal</sub> .

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

TEM (suffixes)

	name: Transmission electron microscopy

	description:
	>Transmission electron microscopy imaging data

TaskDescription (metadata)

	name: TaskDescription

	description:
	>Longer description of the task.

	schema information:
	```yaml
	{'type': 'string'}
	```

TaskName (metadata)

	name: TaskName

	description:
	>Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this `"TaskName"` field by removing all non-alphanumeric (`[a-zA-Z0-9]`) characters. For example `"TaskName"` `"faces n-back"` will correspond to task label `facesnback`.

	schema information:
	```yaml
	{'type': 'string'}
	```

TermURL (metadata)

	name: TermURL

	description:
	>URL pointing to a formal definition of this type of data in an ontology available on the web.

	schema information:
	```yaml
	{'type': 'string'}
	```

TimeZero (metadata)

	name: TimeZero

	description:
	>Time zero to which all scan and/or blood measurements have been adjusted to, in the unit "hh:mm:ss". This should be equal to `"InjectionStart"` or `"ScanStart"`.

	schema information:
	```yaml
	{'type': 'string', 'format': 'time'}
	```

TissueDeformationScaling (metadata)

	name: TissueDeformationScaling

	description:
	>Estimated deformation of the tissue, given as a percentage of the original tissue size (for examples: for a shrinkage of 3%, the value is `97`; and for an expansion of 100%, the value is `200`).

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0}
	```

TissueOrigin (metadata)

	name: TissueOrigin

	description:
	>Describes the type of tissue analyzed for `"SampleOrigin"` `brain`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['gray matter', 'white matter', 'csf', 'meninges', 'macrovascular', 'microvascular']}
	```

TotalAcquiredPairs (metadata)

	name: TotalAcquiredPairs

	description:
	>The total number of acquired `control`-`label` pairs. A single pair consists of a single `control` and a single `label` image.

	schema information:
	```yaml
	{'type': 'number', 'exclusiveMinimum': 0}
	```

TotalReadoutTime (metadata)

	name: TotalReadoutTime

	description:
	>This is actually the "effective" total readout time, defined as the readout duration, specified in seconds, that would have generated data with the given level of distortion. It is NOT the actual, physical duration of the readout train. If `"EffectiveEchoSpacing"` has been properly computed, it is just `EffectiveEchoSpacing * (ReconMatrixPE - 1)`.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

TracerMolecularWeight (metadata)

	name: TracerMolecularWeight

	description:
	>Accurate molecular weight of the tracer used.

	schema information:
	```yaml
	{'type': 'number'}
	```

TracerMolecularWeightUnits (metadata)

	name: TracerMolecularWeightUnits

	description:
	>Unit of the molecular weights measurement (for example, `"g/mol"`).

	schema information:
	```yaml
	{'type': 'string', 'format': 'unit'}
	```

TracerName (metadata)

	name: TracerName

	description:
	>Name of the tracer compound used (for example, `"CIMBI-36"`)

	schema information:
	```yaml
	{'type': 'string'}
	```

TracerRadLex (metadata)

	name: TracerRadLex

	description:
	>ID of the tracer compound from the RadLex Ontology.

	schema information:
	```yaml
	{'type': 'string'}
	```

TracerRadionuclide (metadata)

	name: TracerRadionuclide

	description:
	>Radioisotope labelling tracer (for example, `"C11"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

TracerSNOMED (metadata)

	name: TracerSNOMED

	description:
	>ID of the tracer compound from the SNOMED Ontology (subclass of Radioactive isotope).

	schema information:
	```yaml
	{'type': 'string'}
	```

TriggerChannelCount (metadata)

	name: TriggerChannelCount

	description:
	>Number of channels for digital (TTL bit level) triggers.

	schema information:
	```yaml
	{'type': 'integer', 'minimum': 0}
	```

TubingLength (metadata)

	name: TubingLength

	description:
	>The length of the blood tubing, from the subject to the detector in meters.

	schema information:
	```yaml
	{'type': 'number', 'unit': 'm'}
	```

TubingType (metadata)

	name: TubingType

	description:
	>Description of the type of tubing used, ideally including the material and (internal) diameter.

	schema information:
	```yaml
	{'type': 'string'}
	```

Type (metadata)

	name: Type

	description:
	>Short identifier of the mask. The value `"Brain"` refers to a brain mask. The value `"Lesion"` refers to a lesion mask. The value `"Face"` refers to a face mask. The value `"ROI"` refers to a region of interest mask.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['Brain', 'Lesion', 'Face', 'ROI']}
	```

UNIT1 (suffixes)

	name: Homogeneous (flat) T1-weighted MP2RAGE image

	description:
	>In arbitrary units (arbitrary). UNIT1 images are REQUIRED to use this suffix regardless of the method used to generate them. Note that although this image is T1-weighted, regions without MR signal will contain white salt-and-pepper noise that most segmentation algorithms will fail on. Therefore, it is important to dissociate it from `T1w`. Please see [`MP2RAGE` specific notes](/99-appendices/11-qmri.html#unit1-images) in the qMRI appendix for further information.

Units (metadata)

	name: Units

	description:
	>Measurement units for the associated file. SI units in CMIXF formatting are RECOMMENDED (see [Units](/02-common-principles.html#units)).

	schema information:
	```yaml
	{'type': 'string', 'format': 'unit'}
	```

VFA (suffixes)

	name: Variable flip angle

	description:
	>The VFA method involves at least two spoiled gradient echo (SPGR) of steady-state free precession (SSFP) images acquired at different flip angles. Depending on the provided metadata fields and the sequence type, data may be eligible for DESPOT1, DESPOT2 and their variants ([Deoni et al. 2005](https://doi.org/10.1002/mrm.20314)).

VascularCrushing (metadata)

	name: VascularCrushing

	description:
	>Boolean indicating if Vascular Crushing is used. Corresponds to DICOM Tag 0018, 9259 `ASL Crusher Flag`.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

VascularCrushingVENC (metadata)

	name: VascularCrushingVENC

	description:
	>The crusher gradient strength, in centimeters per second. Specify either one number for the total time-series, or provide an array of numbers, for example when using QUASAR, using the value zero to identify volumes for which `VascularCrushing` was turned off. Corresponds to DICOM Tag 0018, 925A `ASL Crusher Flow Limit`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 'cm/s'}, {'type': 'array', 'items': {'type': 'number', 'unit': 'cm/s'}}]}
	```

VolumeTiming (metadata)

	name: VolumeTiming

	description:
	>The time at which each volume was acquired during the acquisition. It is described using a list of times referring to the onset of each volume in the BOLD series. The list must have the same length as the BOLD series, and the values must be non-negative and monotonically increasing. This field is mutually exclusive with `"RepetitionTime"` and `"DelayTime"`. If defined, this requires acquisition time (TA) be defined via either `"SliceTiming"` or `"AcquisitionDuration"` be defined.

	schema information:
	```yaml
	{'type': 'array', 'minItems': 1, 'items': {'type': 'number', 'unit': 's'}}
	```

WholeBloodAvail (metadata)

	name: WholeBloodAvail

	description:
	>Boolean that specifies if whole blood measurements are available. If `true`, the `whole_blood_radioactivity` column MUST be present in the corresponding `*_blood.tsv` file.

	schema information:
	```yaml
	{'type': 'boolean'}
	```

WithdrawalRate (metadata)

	name: WithdrawalRate

	description:
	>The rate at which the blood was withdrawn from the subject. The unit of the specified withdrawal rate should be in `"mL/s"`.

	schema information:
	```yaml
	{'type': 'number', 'unit': 'mL/s'}
	```

abbreviation (columns)

	name: abbreviation

	description:
	>The unique label abbreviation

	schema information:
	```yaml
	{'type': 'string'}
	```

acq_time _sense 1_ (columns)

	name: acq_time

	description:
	>Acquisition time refers to when the first data point in each run was acquired. Furthermore, if this header is provided, the acquisition times of all files from the same recording MUST be identical. Datetime format and their anonymization are described in [Units](/02-common-principles.html#units).

	schema information:
	```yaml
	{'type': 'string', 'format': 'datetime'}
	```

acq_time _sense 2_ (columns)

	name: acq_time

	description:
	>Acquisition time refers to when the first data point of the first run was acquired. Datetime format and their anonymization are described in [Units](/02-common-principles.html#units).

	schema information:
	```yaml
	{'type': 'string', 'format': 'datetime'}
	```

acquisition (entities)

	name: Acquisition

	description:
	>The `acq-<label>` key/value pair corresponds to a custom label the user MAY use to distinguish a different set of parameters used for acquiring the same modality. For example this should be used when a study includes two T1w images - one full brain low resolution and one restricted field of view but high resolution. In such case two files could have the following names: `sub-01_acq-highres_T1w.nii.gz` and `sub-01_acq-lowres_T1w.nii.gz`, however the user is free to choose any other label than `highres` and `lowres` as long as they are consistent across subjects and sessions. In case different sequences are used to record the same modality (for example, `RARE` and `FLASH` for T1w) this field can also be used to make that distinction. At what level of detail to make the distinction (for example, just between `RARE` and `FLASH`, or between `RARE`, `FLASH`, and `FLASHsubsampled`) remains at the discretion of the researcher.

	schema information:
	```yaml
	{'entity': 'acq', 'type': 'string', 'format': 'label'}
	```

age (columns)

	name: age

	description:
	>Numeric value in years (float or integer value).

	schema information:
	```yaml
	{'type': 'number', 'unit': 'year'}
	```

anat (datatypes)

	name: Anatomical Magnetic Resonance Imaging

	description:
	>Magnetic resonance imaging sequences designed to characterize static, anatomical features.

angio (suffixes)

	name: Angiogram

	description:
	>Magnetic resonance angiography sequences focus on enhancing the contrast of blood vessels (generally arteries, but sometimes veins) against other tissue types.

asl (suffixes)

	name: Arterial Spin Labeling

	description:
	>The complete ASL time series stored as a 4D NIfTI file in the original acquisition order, with possible volume types including: control, label, m0scan, deltam, cbf.

aslcontext (suffixes)

	name: Arterial Spin Labeling Context

	description:
	>A TSV file defining the image types for volumes in an associated ASL file.

asllabeling (suffixes)

	name: ASL Labeling Screenshot

	description:
	>An anonymized screenshot of the planning of the labeling slab/plane with respect to the imaging slab or slices `*_asllabeling.jpg`. Based on DICOM macro C.8.13.5.14.

atlas (entities)

	name: Atlas

	description:
	>The `atlas-<label>` key/value pair corresponds to a custom label the user MAY use to distinguish a different atlas used for similar type of data.<br>This entity is only applicable to derivative data.

	schema information:
	```yaml
	{'entity': 'atlas', 'type': 'string', 'format': 'label'}
	```

beh (datatypes)

	name: Behavioral Data

	description:
	>Behavioral data.

beh (modalities)

	name: Behavioral experiments

	description:
	>Behavioral data acquired without accompanying neuroimaging data.

beh (suffixes)

	name: Behavioral recording

	description:
	>Behavioral recordings from tasks. These files are similar to events files, but do not include the `"onset"` and `"duration"` columns that are mandatory for events files.

bids_uri (formats)

	name: BIDS uniform resource indicator

	description:
	>A BIDS uniform resource indicator.<br>The validation for this format is minimal. It simply ensures that the value is a string with any characters that may appear in a valid URI, starting with "bids:".

	schema information:
	```yaml
	{'pattern': 'bids:[0-9a-zA-Z/#:\\?\\_\\-\\.]+'}
	```

blood (suffixes)

	name: Blood recording data

	description:
	>Blood measurements of radioactivity stored in [tabular files](/02-common-principles.html#tabular-files) and located in the `pet/` directory along with the corresponding PET data.

bold (suffixes)

	name: Blood-Oxygen-Level Dependent image

	description:
	>Blood-Oxygen-Level Dependent contrast (specialized T2\* weighting)

boolean (formats)

	name: Boolean

	description:
	>A boolean. Must be either "true" or "false".

	schema information:
	```yaml
	{'pattern': '(true|false)'}
	```

cardiac (columns)

	name: cardiac

	description:
	>continuous pulse measurement

	schema information:
	```yaml
	{'type': 'number'}
	```

cbv (suffixes)

	name: Cerebral blood volume image

	description:
	>Cerebral Blood Volume contrast (specialized T2\* weighting or difference between T1 weighted images)

ceagent (entities)

	name: Contrast Enhancing Agent

	description:
	>The `ce-<label>` key/value can be used to distinguish sequences using different contrast enhanced images. The label is the name of the contrast agent. The key `"ContrastBolusIngredient"` MAY also be added in the JSON file, with the same label.

	schema information:
	```yaml
	{'entity': 'ce', 'type': 'string', 'format': 'label'}
	```

channels (suffixes)

	name: Channels File

	description:
	>Channel information.

chunk (entities)

	name: Chunk

	description:
	>The `chunk-<index>` key/value pair is used to distinguish between different regions, 2D images or 3D volumes files, of the same physical sample with different fields of view acquired in the same imaging experiment.

	schema information:
	```yaml
	{'entity': 'chunk', 'type': 'string', 'format': 'index'}
	```

code (associated_data)

	name: Code

	description:
	>A directory in which to store any code used to generate the derivatives from the source data.

color (columns)

	name: color

	description:
	>Hexadecimal. Label color for visualization.

	schema information:
	```yaml
	{'type': 'string', 'unit': 'hexadecimal'}
	```

coordsystem (suffixes)

	name: Coordinate System File

	description:
	>A JSON document specifying the coordinate system(s) used for the MEG, EEG, head localization coils, and anatomical landmarks.

dataset_description (top_level_files)

	name: Dataset Description

	description:
	>The file `dataset_description.json` is a JSON file describing the dataset.

dataset_relative (formats)

	name: Path relative to the BIDS dataset directory

	description:
	>A path to a file, relative to the dataset directory.<br>The validation for this format is minimal. It simply ensures that the value is a string with any characters that may appear in a valid path, without starting with "/" (an absolute path).

	schema information:
	```yaml
	{'pattern': '(?!/)[0-9a-zA-Z/\\_\\-\\.]+'}
	```

date (formats)

	name: Date

	description:
	>A date in the form `"YYYY-MM-DD[Z]"`, where [Z] is an optional, valid timezone code.

	schema information:
	```yaml
	{'pattern': '[0-9]{4}-[0-9]{2}-[0-9]{2}([A-Z]{2,4})?'}
	```

datetime (formats)

	name: Datetime

	description:
	>A datetime in the form `"YYYY-MM-DDThh:mm:ss[.000000][Z]"`, where [.000000] is an optional subsecond resolution between 1 and 6 decimal points, and [Z] is an optional, valid timezone code.

	schema information:
	```yaml
	{'pattern': '[0-9]{4}-[0-9]{2}-[0-9]{2}T(?:2[0-3]|[01][0-9]):[0-5][0-9]:[0-5][0-9](\\.[0-9]{1,6})?([A-Z]{2,4})?'}
	```

defacemask (suffixes)

	name: Defacing Mask

	description:
	>A binary mask that was used to remove facial features from an anatomical MRI image.

density (entities)

	name: Density

	description:
	>Density of non-parametric surfaces. MUST have a corresponding `Density` metadata field to provide interpretation.<br>This entity is only applicable to derivative data.

	schema information:
	```yaml
	{'entity': 'den', 'type': 'string', 'format': 'label'}
	```

derivatives (associated_data)

	name: Derivative data

	description:
	>Derivative data

derived_from (columns)

	name: derived_from

	description:
	>`sample-<label>` key/value pair from which a sample is derived, for example a slice of tissue (`sample-02`) derived from a block of tissue (`sample-01`).

	schema information:
	```yaml
	{'type': 'string', 'pattern': '^sample-[0-9a-zA-Z]+$'}
	```

description (columns)

	name: description

	description:
	>Brief free-text description of the channel, or other information of interest.

	schema information:
	```yaml
	{'type': 'string'}
	```

description (entities)

	name: Description

	description:
	>When necessary to distinguish two files that do not otherwise have a distinguishing entity, the `_desc-<label>` keyword-value SHOULD be used.<br>This entity is only applicable to derivative data.

	schema information:
	```yaml
	{'entity': 'desc', 'type': 'string', 'format': 'label'}
	```

dimension (columns)

	name: dimension

	description:
	>Size of the group (grid/strip/probe) that this electrode belongs to. Must be of form `[AxB]` with the smallest dimension first (for example, `[1x8]`).

	schema information:
	```yaml
	{'type': 'string'}
	```

direction (entities)

	name: Phase-Encoding Direction

	description:
	>The `dir-<label>` key/value can be set to an arbitrary alphanumeric label (for example, `dir-LR` or `dir-AP`) to distinguish different phase-encoding directions.

	schema information:
	```yaml
	{'entity': 'dir', 'type': 'string', 'format': 'label'}
	```

duration (columns)

	name: duration

	description:
	>Duration of the event (measured from onset) in seconds. Must always be either zero or positive (or `n/a` if unavailable). A "duration" value of zero implies that the delta function or event is so short as to be effectively modeled as an impulse.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 's', 'minimum': 0}, {'type': 'string', 'enum': ['n/a']}]}
	```

dwi (datatypes)

	name: Diffusion-Weighted Imaging

	description:
	>Diffusion-weighted imaging (DWI).

dwi (suffixes)

	name: Diffusion-weighted image

	description:
	>Diffusion-weighted imaging contrast (specialized T2 weighting).

echo (entities)

	name: Echo

	description:
	>If files belonging to an entity-linked file collection are acquired at different echo times, the `_echo-<index>` key/value pair MUST be used to distinguish individual files. This entity represents the `"EchoTime"` metadata field. Please note that the `<index>` denotes the number/index (in the form of a nonnegative integer), not the `"EchoTime"` value which needs to be stored in the field `"EchoTime"` of the separate JSON file.

	schema information:
	```yaml
	{'entity': 'echo', 'type': 'string', 'format': 'index'}
	```

eeg (datatypes)

	name: Electroencephalography

	description:
	>Electroencephalography

eeg (modalities)

	name: Electroencephalography

	description:
	>Data acquired with EEG.

eeg (suffixes)

	name: Electroencephalography

	description:
	>Electroencephalography recording data.

electrodes (suffixes)

	name: Electrodes

	description:
	>File that gives the location of (i)EEG electrodes.

epi (suffixes)

	name: EPI

	description:
	>The phase-encoding polarity (PEpolar) technique combines two or more Spin Echo EPI scans with different phase encoding directions to estimate the underlying inhomogeneity/deformation map.

events (suffixes)

	name: Events

	description:
	>Event timing information from a behavioral task.

fieldmap (suffixes)

	name: Fieldmap

	description:
	>Some MR schemes such as spiral-echo imaging (SEI) sequences are able to directly provide maps of the *B<sub>0</sub>* field inhomogeneity.

file_relative (formats)

	name: Path relative to the parent file

	description:
	>A path to a file, relative to the file in which the field is defined.<br>The validation for this format is minimal. It simply ensures that the value is a string with any characters that may appear in a valid path, without starting with "/" (an absolute path).

	schema information:
	```yaml
	{'pattern': '(?!/)[0-9a-zA-Z/\\_\\-\\.]+'}
	```

filename (columns)

	name: filename

	description:
	>Relative paths to files.

	schema information:
	```yaml
	{'type': 'string', 'format': 'participant_relative'}
	```

flip (entities)

	name: Flip Angle

	description:
	>If files belonging to an entity-linked file collection are acquired at different flip angles, the `_flip-<index>` key/value pair MUST be used to distinguish individual files. This entity represents the `"FlipAngle"` metadata field. Please note that the `<index>` denotes the number/index (in the form of a nonnegative integer), not the `"FlipAngle"` value which needs to be stored in the field `"FlipAngle"` of the separate JSON file.

	schema information:
	```yaml
	{'entity': 'flip', 'type': 'string', 'format': 'index'}
	```

fmap (datatypes)

	name: Field maps

	description:
	>MRI scans for estimating B0 inhomogeneity-induced distortions.

func (datatypes)

	name: Task-Based Magnetic Resonance Imaging

	description:
	>Task (including resting state) imaging data

genetic_info (top_level_files)

	name: Genetic Information

	description:
	>The `genetic_info.json` file describes the genetic information available in the `participants.tsv` file and/or the genetic database described in `dataset_description.json`. Datasets containing the `Genetics` field in `dataset_description.json` or the `genetic_id` column in `participants.tsv` MUST include this file.

group (columns)

	name: group

	description:
	>Which group of channels (grid/strip/seeg/depth) this channel belongs to. This is relevant because one group has one cable-bundle and noise can be shared. This can be a name or number.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'number'}]}
	```

handedness (columns)

	name: handedness

	description:
	>String value indicating one of "left", "right", "ambidextrous".<br>For "left", use one of these values: `left`, `l`, `L`, `LEFT`, `Left`.<br>For "right", use one of these values: `right`, `r`, `R`, `RIGHT`, `Right`.<br>For "ambidextrous", use one of these values: `ambidextrous`, `a`, `A`, `AMBIDEXTROUS`, `Ambidextrous`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['left', 'l', 'L', 'LEFT', 'Left', 'right', 'r', 'R', 'RIGHT', 'Right', 'ambidextrous', 'a', 'A', 'AMBIDEXTROUS', 'Ambidextrous', 'n/a']}
	```

headshape (suffixes)

	name: Headshape File

	description:
	>The 3-D locations of points that describe the head shape and/or electrode locations can be digitized and stored in separate files.

hemisphere (columns)

	name: hemisphere

	description:
	>The hemisphere in which the electrode is placed.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['L', 'R']}
	```

hemisphere (entities)

	name: Hemisphere

	description:
	>The `hemi-<label>` entity indicates which hemibrain is described by the file. Allowed label values for this entity are `L` and `R`, for the left and right hemibrains, respectively.

	schema information:
	```yaml
	{'entity': 'hemi', 'type': 'string', 'format': 'label', 'enum': ['L', 'R']}
	```

high_cutoff (columns)

	name: high_cutoff

	description:
	>Frequencies used for the low-pass filter applied to the channel in Hz. If no low-pass filter applied, use `n/a`. Note that hardware anti-aliasing in A/D conversion of all MEG/EEG electronics applies a low-pass filter; specify its frequency here if applicable.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 'Hz', 'minimum': 0}, {'type': 'string', 'enum': ['n/a']}]}
	```

hplc_recovery_fractions (columns)

	name: hplc_recovery_fractions

	description:
	>HPLC recovery fractions (the fraction of activity that gets loaded onto the HPLC).

	schema information:
	```yaml
	{'type': 'number', 'unit': 'arbitrary'}
	```

iEEGCoordinateProcessingDescription (metadata)

	name: iEEGCoordinateProcessingDescription

	description:
	>Has any post-processing (such as projection) been done on the electrode positions (for example, `"surface_projection"`, `"none"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

iEEGCoordinateProcessingReference (metadata)

	name: iEEGCoordinateProcessingReference

	description:
	>A reference to a paper that defines in more detail the method used to localize the electrodes and to post-process the electrode positions.

	schema information:
	```yaml
	{'type': 'string'}
	```

iEEGCoordinateSystem (metadata)

	name: iEEGCoordinateSystem

	description:
	>Defines the coordinate system for the iEEG sensors. See [Appendix VIII](/99-appendices/08-coordinate-systems.html) for a list of restricted keywords for coordinate systems. If `"Other"`, provide definition of the coordinate system in `iEEGCoordinateSystemDescription`. If positions correspond to pixel indices in a 2D image (of either a volume-rendering, surface-rendering, operative photo, or operative drawing), this MUST be `"Pixels"`. For more information, see the section on [2D coordinate systems](/04-modality-specific-files/04-intracranial-electroencephalography.html#allowed-2d-coordinate-systems).

	schema information:
	```yaml
	{'type': 'string', 'enum': ['Pixels', 'ACPC', 'Other', 'ICBM452AirSpace', 'ICBM452Warp5Space', 'IXI549Space', 'fsaverage', 'fsaverageSym', 'fsLR', 'MNIColin27', 'MNI152Lin', 'MNI152NLin2009aSym', 'MNI152NLin2009bSym', 'MNI152NLin2009cSym', 'MNI152NLin2009aAsym', 'MNI152NLin2009bAsym', 'MNI152NLin2009cAsym', 'MNI152NLin6Sym', 'MNI152NLin6ASym', 'MNI305', 'NIHPD', 'OASIS30AntsOASISAnts', 'OASIS30Atropos', 'Talairach', 'UNCInfant', 'fsaverage3', 'fsaverage4', 'fsaverage5', 'fsaverage6', 'fsaveragesym', 'UNCInfant0V21', 'UNCInfant1V21', 'UNCInfant2V21', 'UNCInfant0V22', 'UNCInfant1V22', 'UNCInfant2V22', 'UNCInfant0V23', 'UNCInfant1V23', 'UNCInfant2V23']}
	```

iEEGCoordinateSystemDescription (metadata)

	name: iEEGCoordinateSystemDescription

	description:
	>Free-form text description of the coordinate system. May also include a link to a documentation page or paper describing the system in greater detail.

	schema information:
	```yaml
	{'type': 'string'}
	```

iEEGCoordinateUnits (metadata)

	name: iEEGCoordinateUnits

	description:
	>Units of the `*_electrodes.tsv`. MUST be `"pixels"` if `iEEGCoordinateSystem` is `Pixels`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['m', 'mm', 'cm', 'pixels', 'n/a']}
	```

iEEGElectrodeGroups (metadata)

	name: iEEGElectrodeGroups

	description:
	>Field to describe the way electrodes are grouped into strips, grids or depth probes. For example, `"grid1: 10x8 grid on left temporal pole, strip2: 1x8 electrode strip on xxx"`.

	schema information:
	```yaml
	{'type': 'string'}
	```

iEEGGround (metadata)

	name: iEEGGround

	description:
	>Description of the location of the ground electrode (`"placed on right mastoid (M2)"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

iEEGPlacementScheme (metadata)

	name: iEEGPlacementScheme

	description:
	>Freeform description of the placement of the iEEG electrodes. Left/right/bilateral/depth/surface (for example, `"left frontal grid and bilateral hippocampal depth"` or `"surface strip and STN depth"` or `"clinical indication bitemporal, bilateral temporal strips and left grid"`).

	schema information:
	```yaml
	{'type': 'string'}
	```

iEEGReference (metadata)

	name: iEEGReference

	description:
	>General description of the reference scheme used and (when applicable) of location of the reference electrode in the raw recordings (for example, `"left mastoid"`, `"bipolar"`, `"T01"` for electrode with name T01, `"intracranial electrode on top of a grid, not included with data"`, `"upside down electrode"`). If different channels have a different reference, this field should have a general description and the channel specific reference should be defined in the `channels.tsv` file.

	schema information:
	```yaml
	{'type': 'string'}
	```

ieeg (datatypes)

	name: Intracranial electroencephalography

	description:
	>Intracranial electroencephalography (iEEG) or electrocorticography (ECoG) data

ieeg (modalities)

	name: Intracranial Electroencephalography

	description:
	>Data acquired with iEEG.

ieeg (suffixes)

	name: Intracranial Electroencephalography

	description:
	>Intracranial electroencephalography recording data.

impedance (columns)

	name: impedance

	description:
	>Impedance of the electrode, units MUST be in `kOhm`.

	schema information:
	```yaml
	{'type': 'number', 'unit': 'kOhm'}
	```

index (columns)

	name: index

	description:
	>The label integer index.

	schema information:
	```yaml
	{'type': 'integer'}
	```

index (formats)

	name: Index

	description:
	>Non-negative, non-zero integers, optionally prefixed with leading zeros for sortability. An index may not be all zeros.

	schema information:
	```yaml
	{'pattern': '[0-9]*[1-9]+[0-9]*'}
	```

inplaneT1 (suffixes)

	name: Inplane T1

	description:
	>In arbitrary units (arbitrary). T1 weighted structural image matched to a functional (task) image.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

inplaneT2 (suffixes)

	name: Inplane T2

	description:
	>In arbitrary units (arbitrary). T2 weighted structural image matched to a functional (task) image.

	schema information:
	```yaml
	{'unit': 'arbitrary'}
	```

integer (formats)

	name: Integer

	description:
	>An integer which may be positive or negative.

	schema information:
	```yaml
	{'pattern': '[+-]?\\d+'}
	```

inversion (entities)

	name: Inversion Time

	description:
	>If files belonging to an entity-linked file collection are acquired at different inversion times, the `_inv-<index>` key/value pair MUST be used to distinguish individual files. This entity represents the `"InversionTime` metadata field. Please note that the `<index>` denotes the number/index (in the form of a nonnegative integer), not the `"InversionTime"` value which needs to be stored in the field `"InversionTime"` of the separate JSON file.

	schema information:
	```yaml
	{'entity': 'inv', 'type': 'string', 'format': 'index'}
	```

label (entities)

	name: Label

	description:
	>Tissue-type label, following a prescribed vocabulary. Applies to binary masks and probabilistic/partial volume segmentations that describe a single tissue type.<br>This entity is only applicable to derivative data.

	schema information:
	```yaml
	{'entity': 'label', 'type': 'string', 'format': 'label'}
	```

label (formats)

	name: Label

	description:
	>Freeform labels without special characters.

	schema information:
	```yaml
	{'pattern': '[0-9a-zA-Z]+'}
	```

low_cutoff (columns)

	name: low_cutoff

	description:
	>Frequencies used for the high-pass filter applied to the channel in Hz. If no high-pass filter applied, use `n/a`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 'Hz'}, {'type': 'string', 'enum': ['n/a']}]}
	```

m0scan (suffixes)

	name: M0 image

	description:
	>The M0 image is a calibration image, used to estimate the equilibrium magnetization of blood.

magnitude (suffixes)

	name: Magnitude

	description:
	>Field-mapping MR schemes such as gradient-recalled echo (GRE) generate a Magnitude image to be used for anatomical reference. Requires the existence of Phase, Phase-difference or Fieldmap maps.

magnitude1 (suffixes)

	name: Magnitude

	description:
	>Magnitude map generated by GRE or similar schemes, associated with the first echo in the sequence.

magnitude2 (suffixes)

	name: Magnitude

	description:
	>Magnitude map generated by GRE or similar schemes, associated with the second echo in the sequence.

manufacturer (columns)

	name: manufacturer

	description:
	>The manufacturer for each electrode. Can be used if electrodes were manufactured by more than one company.

	schema information:
	```yaml
	{'type': 'string'}
	```

mapping (columns)

	name: mapping

	description:
	>Corresponding integer label in the standard BIDS label lookup.

	schema information:
	```yaml
	{'type': 'integer'}
	```

markers (suffixes)

	name: MEG Sensor Coil Positions

	description:
	>Another manufacturer-specific detail pertains to the KIT/Yokogawa/Ricoh system, which saves the MEG sensor coil positions in a separate file with two possible filename extensions  (`.sqd`, `.mrk`). For these files, the `markers` suffix MUST be used. For example: `sub-01_task-nback_markers.sqd`

material (columns)

	name: material

	description:
	>Material of the electrode (for example, `Tin`, `Ag/AgCl`, `Gold`).

	schema information:
	```yaml
	{'type': 'string'}
	```

meg (datatypes)

	name: Magnetoencephalography

	description:
	>Magnetoencephalography

meg (modalities)

	name: Magnetoencephalography

	description:
	>Data acquired with an MEG scanner.

meg (suffixes)

	name: Magnetoencephalography

	description:
	>Unprocessed MEG data stored in the native file format of the MEG instrument with which the data was collected.

metabolite_parent_fraction (columns)

	name: metabolite_parent_fraction

	description:
	>Parent fraction of the radiotracer (0-1).

	schema information:
	```yaml
	{'type': 'number', 'minimum': 0, 'maximum': 1}
	```

metabolite_polar_fraction (columns)

	name: metabolite_polar_fraction

	description:
	>Polar metabolite fraction of the radiotracer (0-1).

	schema information:
	```yaml
	{'type': 'number', 'minimum': 0, 'maximum': 1}
	```

micr (datatypes)

	name: Microscopy

	description:
	>Microscopy

micr (modalities)

	name: Microscopy

	description:
	>Data acquired with a microscope.

modality (entities)

	name: Corresponding Modality

	description:
	>The `mod-<label>` key/value pair corresponds to modality label for defacing masks, for example, T1w, inplaneT1, referenced by a defacemask image. For example, `sub-01_mod-T1w_defacemask.nii.gz`.

	schema information:
	```yaml
	{'entity': 'mod', 'type': 'string', 'format': 'label'}
	```

mri (modalities)

	name: Magnetic Resonance Imaging

	description:
	>Data acquired with an MRI scanner.

mtransfer (entities)

	name: Magnetization Transfer

	description:
	>If files belonging to an entity-linked file collection are acquired at different magnetization transfer (MT) states, the `_mt-<label>` key/value pair MUST be used to distinguish individual files. This entity represents the `"MTState"` metadata field. Allowed label values for this entity are `on` and `off`, for images acquired in presence and absence of an MT pulse, respectively.

	schema information:
	```yaml
	{'entity': 'mt', 'type': 'string', 'enum': ['on', 'off']}
	```

name _sense 1_ (columns)

	name: name

	description:
	>Label of the channel.

	schema information:
	```yaml
	{'type': 'string'}
	```

name _sense 2_ (columns)

	name: name

	description:
	>Name of the electrode contact point.

	schema information:
	```yaml
	{'type': 'string'}
	```

name _sense 3_ (columns)

	name: name

	description:
	>The unique label name.

	schema information:
	```yaml
	{'type': 'string'}
	```

notch (columns)

	name: notch

	description:
	>Frequencies used for the notch filter applied to the channel, in Hz. If no notch filter applied, use `n/a`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 'Hz'}, {'type': 'string', 'enum': ['n/a']}]}
	```

number (formats)

	name: Number

	description:
	>A number which may be an integer or float, positive or negative.

	schema information:
	```yaml
	{'pattern': '[+-]?([0-9]+([.][0-9]*)?|[.][0-9]+)([eE][+-]?[0-9]+)?'}
	```

onset (columns)

	name: onset

	description:
	>Onset (in seconds) of the event, measured from the beginning of the acquisition of the first data point stored in the corresponding task data file. Negative onsets are allowed, to account for events that occur prior to the first stored data point. For example, in case there is an in-scanner training phase that begins before the scanning sequence has started events from this sequence should have negative onset time counting down to the beginning of the acquisition of the first volume.<br>If any data points have been discarded before forming the data file (for example, "dummy volumes" in BOLD fMRI), a time of 0 corresponds to the first stored data point and not the first acquired data point.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

part (entities)

	name: Part

	description:
	>This entity is used to indicate which component of the complex representation of the MRI signal is represented in voxel data. The `part-<label>` key/value pair is associated with the DICOM Tag `0008, 9208`. Allowed label values for this entity are `phase`, `mag`, `real` and `imag`, which are typically used in `part-mag`/`part-phase` or `part-real`/`part-imag` pairs of files.<br>Phase images MAY be in radians or in arbitrary units. The sidecar JSON file MUST include the units of the `phase` image. The possible options are `"rad"` or `"arbitrary"`.<br>When there is only a magnitude image of a given type, the `part` key MAY be omitted.

	schema information:
	```yaml
	{'entity': 'part', 'type': 'string', 'enum': ['mag', 'phase', 'real', 'imag']}
	```

participant_id (columns)

	name: participant_id

	description:
	>A participant identifier of the form `sub-<label>`, matching a participant entity found in the dataset.

	schema information:
	```yaml
	{'type': 'string', 'pattern': '^sub-[0-9a-zA-Z]+$'}
	```

participant_relative (formats)

	name: Path relative to the participant directory

	description:
	>A path to a file, relative to the participant's directory in the dataset.<br>The validation for this format is minimal. It simply ensures that the value is a string with any characters that may appear in a valid path, without starting with "/" (an absolute path) or "sub/" (a relative path starting with the participant directory, rather than relative to that directory).

	schema information:
	```yaml
	{'pattern': '(?!/)(?!sub-)[0-9a-zA-Z/\\_\\-\\.]+'}
	```

participants (top_level_files)

	name: Participant Information

	description:
	>The purpose of this RECOMMENDED file is to describe properties of participants such as age, sex, handedness. If this file exists, it MUST contain the column `participant_id`, which MUST consist of `sub-<label>` values identifying one row for each participant, followed by a list of optional columns describing participants. Each participant MUST be described by one and only one row.<br>Commonly used *optional* columns in `participant.tsv` files are `age`, `sex`, and `handedness`. We RECOMMEND to make use of these columns, and in case that you do use them, we RECOMMEND to use the following values for them:<br>-   `age`: numeric value in years (float or integer value)<br>-   `sex`: string value indicating phenotypical sex, one of "male", "female",     "other"<br>    -   for "male", use one of these values: `male`, `m`, `M`, `MALE`, `Male`<br>    -   for "female", use one of these values: `female`, `f`, `F`, `FEMALE`,         `Female`<br>    -   for "other", use one of these values: `other`, `o`, `O`, `OTHER`,         `Other`<br>-   `handedness`: string value indicating one of "left", "right",     "ambidextrous"<br>    -   for "left", use one of these values: `left`, `l`, `L`, `LEFT`, `Left`<br>    -   for "right", use one of these values: `right`, `r`, `R`, `RIGHT`,         `Right`<br>    -   for "ambidextrous", use one of these values: `ambidextrous`, `a`, `A`,         `AMBIDEXTROUS`, `Ambidextrous`<br>Throughout BIDS you can indicate missing values with `n/a` (for "not available").

pathology (columns)

	name: pathology

	description:
	>String value describing the pathology of the sample or type of control. When different from `healthy`, pathology SHOULD be specified. The pathology may be specified in either `samples.tsv` or `sessions.tsv`, depending on whether the pathology changes over time.

	schema information:
	```yaml
	{'type': 'string'}
	```

perf (datatypes)

	name: Perfusion imaging

	description:
	>Blood perfusion imaging data, including arterial spin labeling (ASL)

pet (datatypes)

	name: Positron Emission Tomography

	description:
	>Positron emission tomography data

pet (modalities)

	name: Positron Emission Tomography

	description:
	>Data acquired with PET.

pet (suffixes)

	name: Positron Emission Tomography

	description:
	>PET imaging data SHOULD be stored in 4D (or 3D, if only one volume was acquired) NIfTI files with the `_pet` suffix. Volumes MUST be stored in chronological order (the order they were acquired in).

phase (suffixes)

	name: Phase image

	description:
	>[DEPRECATED](/02-common-principles.html#definitions). Phase information associated with magnitude information stored in BOLD contrast. This suffix should be replaced by the [`part-phase`](/99-appendices/09-entities.html#part) in conjunction with the `bold` suffix.

	schema information:
	```yaml
	{'anyOf': [{'unit': 'arbitrary'}, {'unit': 'rad'}]}
	```

phase1 (suffixes)

	name: Phase

	description:
	>Phase map generated by GRE or similar schemes, associated with the first echo in the sequence.

phase2 (suffixes)

	name: Phase

	description:
	>Phase map generated by GRE or similar schemes, associated with the second echo in the sequence.

phasediff (suffixes)

	name: Phase-difference

	description:
	>Some scanners subtract the `phase1` from the `phase2` map and generate a unique `phasediff` file. For instance, this is a common output for the built-in fieldmap sequence of Siemens scanners.

photo (suffixes)

	name: Photo File

	description:
	>Photos of the anatomical landmarks, head localization coils or tissue sample.

physio (suffixes)

	name: Physiological recording

	description:
	>Physiological recordings such as cardiac and respiratory signals.

plasma_radioactivity (columns)

	name: plasma_radioactivity

	description:
	>Radioactivity in plasma, in unit of plasma radioactivity (for example, `kBq/mL`).

	schema information:
	```yaml
	{'type': 'number'}
	```

processing (entities)

	name: Processed (on device)

	description:
	>The proc label is analogous to rec for MR and denotes a variant of a file that was a result of particular processing performed on the device.<br>This is useful for files produced in particular by Elekta's MaxFilter (for example, `sss`, `tsss`, `trans`, `quat` or `mc`), which some installations impose to be run on raw data because of active shielding software corrections before the MEG data can actually be exploited.

	schema information:
	```yaml
	{'entity': 'proc', 'type': 'string', 'format': 'label'}
	```

reconstruction (entities)

	name: Reconstruction

	description:
	>The `rec-<label>` key/value can be used to distinguish different reconstruction algorithms (for example `MoCo` for the ones using motion correction).

	schema information:
	```yaml
	{'entity': 'rec', 'type': 'string', 'format': 'label'}
	```

recording (entities)

	name: Recording

	description:
	>More than one continuous recording file can be included (with different sampling frequencies). In such case use different labels. For example: `_recording-contrast`, `_recording-saturation`.

	schema information:
	```yaml
	{'entity': 'recording', 'type': 'string', 'format': 'label'}
	```

reference _sense 1_ (columns)

	name: reference

	description:
	>Name of the reference electrode(s). This column is not needed when it is common to all channels. In that case the reference electrode(s) can be specified in `*_eeg.json` as `EEGReference`).

	schema information:
	```yaml
	{'type': 'string'}
	```

reference _sense 2_ (columns)

	name: reference

	description:
	>Specification of the reference (for example, `mastoid`, `ElectrodeName01`, `intracranial`, `CAR`, `other`, `n/a`). If the channel is not an electrode channel (for example, a microphone channel) use `n/a`.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'string', 'enum': ['n/a']}]}
	```

resolution (entities)

	name: Resolution

	description:
	>Resolution of regularly sampled N-dimensional data. MUST have a corresponding `"Resolution"` metadata field to provide interpretation.<br>This entity is only applicable to derivative data.

	schema information:
	```yaml
	{'entity': 'res', 'type': 'string', 'format': 'label'}
	```

respiratory (columns)

	name: respiratory

	description:
	>continuous breathing measurement

	schema information:
	```yaml
	{'type': 'number'}
	```

response_time (columns)

	name: response_time

	description:
	>Response time measured in seconds. A negative response time can be used to represent preemptive responses and `n/a` denotes a missed response.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number', 'unit': 's'}, {'type': 'string', 'enum': ['n/a']}]}
	```

rrid (formats)

	name: Research resource identifier

	description:
	>A [research resource identifier](https://scicrunch.org/resources).

	schema information:
	```yaml
	{'pattern': 'RRID:.+_.+'}
	```

run (entities)

	name: Run

	description:
	>If several scans with the same acquisition parameters are acquired in the same session, they MUST be indexed with the [`run-<index>`](../99-appendices/09-entities.md#run) entity: `_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as run labels).<br>If different entities apply, such as a different session indicated by [`ses-<label>`](../99-appendices/09-entities.md#ses), or different acquisition parameters indicated by [`acq-<label>`](../99-appendices/09-entities.md#acq), then `run` is not needed to distinguish the scans and MAY be omitted.

	schema information:
	```yaml
	{'entity': 'run', 'type': 'string', 'format': 'index'}
	```

sample (columns)

	name: sample

	description:
	>Onset of the event according to the sampling scheme of the recorded modality (that is, referring to the raw data file that the `events.tsv` file accompanies).

	schema information:
	```yaml
	{'type': 'integer'}
	```

sample (entities)

	name: Sample

	description:
	>A sample pertaining to a subject such as tissue, primary cell or cell-free sample. The `sample-<label>` key/value pair is used to distinguish between different samples from the same subject. The label MUST be unique per subject and is RECOMMENDED to be unique throughout the dataset.

	schema information:
	```yaml
	{'entity': 'sample', 'type': 'string', 'format': 'label'}
	```

sample_id (columns)

	name: sample_id

	description:
	>A sample identifier of the form `sample-<label>`, matching a sample entity found in the dataset.

	schema information:
	```yaml
	{'type': 'string', 'pattern': '^sample-[0-9a-zA-Z]+$'}
	```

sample_type (columns)

	name: sample_type

	description:
	>Biosample type defined by [ENCODE Biosample Type](https://www.encodeproject.org/profiles/biosample_type).

	schema information:
	```yaml
	{'type': 'string', 'enum': ['cell line', 'in vitro differentiated cells', 'primary cell', 'cell-free sample', 'cloning host', 'tissue', 'whole organisms', 'organoid', 'technical sample']}
	```

samples (top_level_files)

	name: Sample Information

	description:
	>The purpose of this file is to describe properties of samples, indicated by the `sample` entity. This file is REQUIRED if `sample-<label>` is present in any file name within the dataset. If this file exists, it MUST contain the three following columns:<br>-   `sample_id`: MUST consist of `sample-<label>` values identifying one row     for each sample<br>-   `participant_id`: MUST consist of `sub-<label>`<br>-   `sample_type`: MUST consist of sample type values, either `cell line`, `in vitro differentiated cells`,     `primary cell`, `cell-free sample`, `cloning host`, `tissue`, `whole organisms`, `organoid` or     `technical sample` from [ENCODE Biosample Type](https://www.encodeproject.org/profiles/biosample_type)<br>Other optional columns MAY be used to describe the samples. Each sample MUST be described by one and only one row.<br>Commonly used *optional* columns in `samples.tsv` files are `pathology` and `derived_from`. We RECOMMEND to make use of these columns, and in case that you do use them, we RECOMMEND to use the following values for them:<br>-   `pathology`: string value describing the pathology of the sample or type of control.     When different from `healthy`, pathology SHOULD be specified in `samples.tsv`.     The pathology MAY instead be specified in     [Sessions files](../03-modality-agnostic-files.md#sessions-file) in case it changes over time.<br>-   `derived_from`: `sample-<label>` key/value pair from which a sample is derived from,     for example a slice of tissue (`sample-02`) derived from a block of tissue (`sample-01`)

sampling_frequency (columns)

	name: sampling_frequency

	description:
	>Sampling rate of the channel in Hz.

	schema information:
	```yaml
	{'type': 'number', 'unit': 'Hz'}
	```

sbref (suffixes)

	name: Single-band reference image

	description:
	>Single-band reference for one or more multi-band `dwi` images.

scans (suffixes)

	name: Scans file

	description:
	>The purpose of this file is to describe timing and other properties of each imaging acquisition sequence (each run file) within one session. Each neural recording file SHOULD be described by exactly one row. Some recordings consist of multiple parts, that span several files, for example through echo-, part-, or split- entities. Such recordings MUST be documented with one row per file. Relative paths to files should be used under a compulsory filename header. If acquisition time is included it should be listed under the acq_time header. Acquisition time refers to when the first data point in each run was acquired. Furthermore, if this header is provided, the acquisition times of all files that belong to a recording MUST be identical. Datetime should be expressed as described in Units. Additional fields can include external behavioral measures relevant to the scan. For example vigilance questionnaire score administered after a resting state scan. All such included additional fields SHOULD be documented in an accompanying _scans.json file that describes these fields in detail (see Tabular files).

session (entities)

	name: Session

	description:
	>A logical grouping of neuroimaging and behavioral data consistent across subjects. Session can (but doesn't have to) be synonymous to a visit in a longitudinal study. In general, subjects will stay in the scanner during one session. However, for example, if a subject has to leave the scanner room and then be re-positioned on the scanner bed, the set of MRI acquisitions will still be considered as a session and match sessions acquired in other subjects. Similarly, in situations where different data types are obtained over several visits (for example fMRI on one day followed by DWI the day after) those can be grouped in one session. Defining multiple sessions is appropriate when several identical or similar data acquisitions are planned and performed on all -or most- subjects, often in the case of some intervention between sessions (for example, training).

	schema information:
	```yaml
	{'entity': 'ses', 'type': 'string', 'format': 'label'}
	```

session_id (columns)

	name: session_id

	description:
	>A session identifier of the form `ses-<label>`, matching a session found in the dataset.

	schema information:
	```yaml
	{'type': 'string', 'pattern': '^ses-[0-9a-zA-Z]+$'}
	```

sessions (suffixes)

	name: Sessions file

	description:
	>In case of multiple sessions there is an option of adding additional sessions.tsv files describing variables changing between sessions. In such case one file per participant SHOULD be added. These files MUST include a session_id column and describe each session by one and only one row. Column names in sessions.tsv files MUST be different from group level participant key column names in the participants.tsv file.

sex (columns)

	name: sex

	description:
	>String value indicating phenotypical sex, one of "male", "female", "other".<br>For "male", use one of these values: `male`, `m`, `M`, `MALE`, `Male`.<br>For "female", use one of these values: `female`, `f`, `F`, `FEMALE`, `Female`.<br>For "other", use one of these values: `other`, `o`, `O`, `OTHER`, `Other`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['male', 'm', 'M', 'MALE', 'Male', 'female', 'f', 'F', 'FEMALE', 'Female', 'other', 'o', 'O', 'OTHER', 'Other', 'n/a']}
	```

size (columns)

	name: size

	description:
	>Surface area of the electrode, units MUST be in `mm^2`.

	schema information:
	```yaml
	{'type': 'number', 'unit': 'mm2'}
	```

software_filters (columns)

	name: software_filters

	description:
	>List of temporal and/or spatial software filters applied (for example, `SSS`, `SpatialCompensation`). Note that parameters should be defined in the general MEG sidecar .json file. Indicate `n/a` in the absence of software filters applied.

	schema information:
	```yaml
	{'anyOf': [{'type': 'string'}, {'type': 'string', 'enum': ['n/a']}]}
	```

sourcedata (associated_data)

	name: Source data

	description:
	>Data before harmonization, reconstruction, and/or file format conversion (for example, E-Prime event logs or DICOM files).

space (entities)

	name: Space

	description:
	>The space entity can be used to indicate the way in which electrode positions are interpreted (for EEG/MEG/iEEG data) or the spatial reference to which a file has been aligned (for MRI data). The space `<label>` MUST be taken from one of the modality specific lists in [Appendix VIII](../99-appendices/08-coordinate-systems.md). For example for iEEG data, the restricted keywords listed under [iEEG Specific Coordinate Systems](../99-appendices/08-coordinate-systems.md#ieeg-specific-coordinate-systems) are acceptable for `<label>`.<br>For EEG/MEG/iEEG data, this entity can be applied to raw data, but for other data types, it is restricted to derivative data.

	schema information:
	```yaml
	{'entity': 'space', 'type': 'string', 'format': 'label'}
	```

species (columns)

	name: species

	description:
	>The `species` column SHOULD be a binomial species name from the [NCBI Taxonomy](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi) (for example, `homo sapiens`, `mus musculus`, `rattus norvegicus`). For backwards compatibility, if `species` is absent, the participant is assumed to be `homo sapiens`.

	schema information:
	```yaml
	{'type': 'string'}
	```

split (entities)

	name: Split

	description:
	>In the case of long data recordings that exceed a file size of 2Gb, the .fif files are conventionally split into multiple parts. Each of these files has an internal pointer to the next file. This is important when renaming these split recordings to the BIDS convention.<br>Instead of a simple renaming, files should be read in and saved under their new names with dedicated tools like [MNE-Python](https://mne.tools/), which will ensure that not only the file names, but also the internal file pointers will be updated. It is RECOMMENDED that .fif files with multiple parts use the `split-<index>` entity to indicate each part. If there are multiple parts of a recording and the optional `scans.tsv` is provided, remember to list all files separately in `scans.tsv` and that the entries for the `acq_time` column in `scans.tsv` MUST all be identical, as described in [Scans file](../03-modality-agnostic-files.md#scans-file).

	schema information:
	```yaml
	{'entity': 'split', 'type': 'string', 'format': 'index'}
	```

stain (entities)

	name: Stain

	description:
	>The `stain-<label>` key/pair values can be used to distinguish image files from the same sample using different stains or antibodies for contrast enhancement. Stains SHOULD be indicated in the `"SampleStaining"` key in the sidecar JSON file, although the label may be different. Description of antibodies SHOULD also be indicated in `"SamplePrimaryAntibodies"` and/or `"SampleSecondaryAntobodies"` as appropriate.

	schema information:
	```yaml
	{'entity': 'stain', 'type': 'string', 'format': 'label'}
	```

status (columns)

	name: status

	description:
	>Data quality observed on the channel. A channel is considered `bad` if its data quality is compromised by excessive noise. If quality is unknown, then a value of `n/a` may be used. Description of noise type SHOULD be provided in `[status_description]`.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['good', 'bad', 'n/a']}
	```

status_description (columns)

	name: status_description

	description:
	>Freeform text description of noise or artifact affecting data quality on the channel. It is meant to explain why the channel was declared bad in `[status]`.

	schema information:
	```yaml
	{'type': 'string'}
	```

stim (suffixes)

	name: Continuous recording

	description:
	>Continuous measures, such as parameters of a film or audio stimulus.

stim_file (columns)

	name: stim_file

	description:
	>Represents the location of the stimulus file (such as an image, video, or audio file) presented at the given onset time. There are no restrictions on the file formats of the stimuli files, but they should be stored in the `/stimuli` directory (under the root directory of the dataset; with optional subdirectories). The values under the `stim_file` column correspond to a path relative to `/stimuli`. For example `images/cat03.jpg` will be translated to `/stimuli/images/cat03.jpg`.

	schema information:
	```yaml
	{'type': 'string', 'format': 'stimuli_relative'}
	```

stimuli (associated_data)

	name: Stimulus files

	description:
	>The stimulus files can be added in a `/stimuli` directory (under the root directory of the dataset; with optional subdirectories) AND using a stim_file column in *_events.tsv mentioning which stimulus file was used for a given event.<br>There are no restrictions on the file formats of the stimuli files, but they should be stored in the `/stimuli` directory.

stimuli_relative (formats)

	name: Path relative to the stimuli directory

	description:
	>A path to a stimulus file, relative to a `/stimuli` directory somewhere.<br>The validation for this format is minimal. It simply ensures that the value is a string with any characters that may appear in a valid path, without starting with "/" (an absolute path) or "stimuli/" (a relative path starting with the stimuli directory, rather than relative to that directory).

	schema information:
	```yaml
	{'pattern': '(?!/)(?!stimuli/)[0-9a-zA-Z/\\_\\-\\.]+'}
	```

strain (columns)

	name: strain

	description:
	>For species different from `homo sapiens`, string value indicating the strain of the species, for example: `C57BL/6J`.

	schema information:
	```yaml
	{'type': 'string'}
	```

strain_rrid (columns)

	name: strain_rrid

	description:
	>For species different from `homo sapiens`, research resource identifier ([RRID](https://scicrunch.org/resources/Organisms/search)) of the strain of the species, for example: `RRID:IMSR_JAX:000664`.

	schema information:
	```yaml
	{'type': 'string', 'format': 'rrid'}
	```

string (formats)

	name: String

	description:
	>The basic string type (not a specific format). This should allow any free-form string.

	schema information:
	```yaml
	{'pattern': '.*'}
	```

subject (entities)

	name: Subject

	description:
	>A person or animal participating in the study.

	schema information:
	```yaml
	{'entity': 'sub', 'type': 'string', 'format': 'label'}
	```

task (entities)

	name: Task

	description:
	>Each task has a unique label that MUST only consist of letters and/or numbers (other characters, including spaces and underscores, are not allowed). Those labels MUST be consistent across subjects and sessions.

	schema information:
	```yaml
	{'entity': 'task', 'type': 'string', 'format': 'label'}
	```

time (columns)

	name: time

	description:
	>Time, in seconds, relative to `TimeZero` defined by the `*_pet.json`. For example, 5.

	schema information:
	```yaml
	{'type': 'number', 'unit': 's'}
	```

time (formats)

	name: Time

	description:
	>A time in the form `"hh:mm:ss"`.

	schema information:
	```yaml
	{'pattern': '(?:2[0-3]|[01]?[0-9]):[0-5][0-9]:[0-5][0-9]'}
	```

tracer (entities)

	name: Tracer

	description:
	>The `trc-<label>` key/value can be used to distinguish sequences using different tracers. The key `"TracerName"` MUST also be included in the associated JSON file, although the label may be different.

	schema information:
	```yaml
	{'entity': 'trc', 'type': 'string', 'format': 'label'}
	```

trial_type (columns)

	name: trial_type

	description:
	>Primary categorisation of each trial to identify them as instances of the experimental conditions. For example: for a response inhibition task, it could take on values `go` and `no-go` to refer to response initiation and response inhibition experimental conditions.

	schema information:
	```yaml
	{'type': 'string'}
	```

trigger (columns)

	name: trigger

	description:
	>continuous measurement of the scanner trigger signal

	schema information:
	```yaml
	{'type': 'number'}
	```

type _sense 1_ (columns)

	name: type

	description:
	>Type of channel; MUST use the channel types listed below. Note that the type MUST be in upper-case.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['MEGMAG', 'MEGGRADAXIAL', 'MEGGRADPLANAR', 'MEGREFMAG', 'MEGREFGRADAXIAL', 'MEGREFGRADPLANAR', 'MEGOTHER', 'EEG', 'ECOG', 'SEEG', 'DBS', 'VEOG', 'HEOG', 'EOG', 'ECG', 'EMG', 'TRIG', 'AUDIO', 'PD', 'EYEGAZE', 'PUPIL', 'MISC', 'SYSCLOCK', 'ADC', 'DAC', 'HLU', 'FITERR', 'OTHER']}
	```

type _sense 2_ (columns)

	name: type

	description:
	>Type of the electrode (for example, cup, ring, clip-on, wire, needle).

	schema information:
	```yaml
	{'type': 'string'}
	```

uCT (suffixes)

	name: Micro-CT

	description:
	>Micro-CT imaging data

unit (formats)

	name: A standardized unit

	description:
	>A unit. SI units in CMIXF formatting are RECOMMENDED (see [Units](/02-common-principles.html#units)).<br>Currently this matches any string.<br>TODO: Somehow reference the actual unit options in the Units appendix.

	schema information:
	```yaml
	{'pattern': '.*'}
	```

units (columns)

	name: units

	description:
	>Physical unit of the value represented in this channel, for example, `V` for Volt, or `fT/cm` for femto Tesla per centimeter (see [Units](/02-common-principles.html#units)).

	schema information:
	```yaml
	{'type': 'string', 'format': 'unit'}
	```

uri (formats)

	name: Uniform resource indicator

	description:
	>A uniform resource indicator.

	schema information:
	```yaml
	{'pattern': '(([^:/?#]+):)?(//([^/?#]*))?([^?#]*)(\\?([^#]*))?(#(.*))?'}
	```

value (columns)

	name: value

	description:
	>Marker value associated with the event (for example, the value of a TTL trigger that was recorded at the onset of the event).

	schema information:
	```yaml
	{'anyOf': [{'type': 'number'}, {'type': 'string'}]}
	```

volume_type (columns)

	name: volume_type

	description:
	>The `*_aslcontext.tsv` table consists of a single column of labels identifying the `volume_type` of each volume in the corresponding `*_asl.nii[.gz]` file.

	schema information:
	```yaml
	{'type': 'string', 'enum': ['control', 'label', 'm0scan', 'deltam', 'cbf']}
	```

whole_blood_radioactivity (columns)

	name: whole_blood_radioactivity

	description:
	>Radioactivity in whole blood samples, in unit of radioactivity measurements in whole blood samples (for example, `kBq/mL`).

	schema information:
	```yaml
	{'type': 'number'}
	```

x (columns)

	name: x

	description:
	>Recorded position along the x-axis.

	schema information:
	```yaml
	{'type': 'number'}
	```

y (columns)

	name: y

	description:
	>Recorded position along the y-axis.

	schema information:
	```yaml
	{'type': 'number'}
	```

z (columns)

	name: z

	description:
	>Recorded position along the z-axis.

	schema information:
	```yaml
	{'anyOf': [{'type': 'number'}, {'type': 'string', 'enum': ['n/a']}]}
	```

```