{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "3e9f27fb",
   "metadata": {},
   "source": [
    "# Genetic Descriptor\n",
    "\n",
    "Support genetic descriptors was developed as a\n",
    "[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).\n",
    "Please see [Citing BIDS](../01-introduction.md#citing-bids)\n",
    "on how to appropriately credit this extension when referring to it in the\n",
    "context of the academic literature.\n",
    "\n",
    "Genetic data are typically stored in dedicated repositories,\n",
    "separate from imaging data.\n",
    "A genetic descriptor links a BIDS dataset to associated genetic data,\n",
    "potentially in a separate repository,\n",
    "with details of where to find the genetic data and the type of data available.\n",
    "\n",
    "The following example dataset with genetics data have been formatted using this specification\n",
    "and can be used for practical guidance when curating a new dataset.\n",
    "\n",
    "-   [`UK biobank`](https://github.com/bids-standard/bids-examples/tree/master/genetics_ukbb)\n",
    "\n",
    "## Dataset Description\n",
    "\n",
    "Genetic descriptors are encoded as an additional, OPTIONAL entry in the\n",
    "[`dataset_description.json`](../03-modality-agnostic-files.md#dataset_descriptionjson)\n",
    "file.\n",
    "\n",
    "Datasets linked to a genetic database entry include the following REQUIRED or OPTIONAL\n",
    "`dataset_description.json` keys (a dot in the key name denotes a key in a sub-[object][],\n",
    "see the example further below):\n",
    "\n",
    "{{ MACROS___make_metadata_table(\n",
    "   {\n",
    "      \"Genetics.Dataset\": \"REQUIRED\",\n",
    "      \"Genetics.Database\": \"OPTIONAL\",\n",
    "      \"Genetics.Descriptors\": \"OPTIONAL\",\n",
    "   }\n",
    ") }}\n",
    "\n",
    "Example:\n",
    "\n",
    "```JSON\n",
    "{\n",
    "  \"Name\": \"Human Connectome Project\",\n",
    "  \"BIDSVersion\":  \"1.3.0\",\n",
    "  \"License\": \"CC0\",\n",
    "  \"Authors\": [\"1st author\", \"2nd author\"],\n",
    "  \"Funding\": [\"P41 EB015894/EB/NIBIB NIH HHS/United States\"],\n",
    "  \"Genetics\": {\n",
    "     \"Dataset\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001364.v1.p1\",\n",
    "     \"Database\": \"https://www.ncbi.nlm.nih.gov/gap/\",\n",
    "     \"Descriptors\": [\"doi:10.1016/j.neuroimage.2013.05.041\"]\n",
    "     }\n",
    "}\n",
    "```\n",
    "\n",
    "## Subject naming and Participants file\n",
    "\n",
    "If the same participants have different identifiers in the genetic and imaging datasets,\n",
    "the column `genetic_id` SHOULD be added to the `participants.tsv` file to associate\n",
    "the BIDS participant with a subject in the `Genetics.Dataset` referred to in the\n",
    "`dataset_description.json` file.\n",
    "\n",
    "Information about the presence/absence of specific genetic markers MAY be duplicated\n",
    "in the `participants.tsv` file by adding optional columns (like `idh_mutation` in the\n",
    "example below).\n",
    "Note that optional columns MUST be further described in an accompanying\n",
    "`participants.json` file as described in\n",
    "[Tabular files](../02-common-principles.md#tabular-files).\n",
    "\n",
    "`participants.tsv` example:\n",
    "\n",
    "```Text\n",
    "participant_id\tage\tsex\tgroup\tgenetic_id\tidh_mutation\n",
    "sub-control01\t34\tM\tcontrol\t124587\tyes\n",
    "sub-control02\t12\tF\tcontrol\t548936\tyes\n",
    "sub-patient01\t33\tF\tpatient\t489634\tno\n",
    "```\n",
    "\n",
    "## Genetic Information\n",
    "\n",
    "Template:\n",
    "\n",
    "```Text\n",
    "genetic_info.json\n",
    "```\n",
    "\n",
    "The `genetic_info.json` file describes the genetic information available in the\n",
    "`participants.tsv` file and/or the genetic database described in\n",
    "`dataset_description.json`.\n",
    "Datasets containing the `Genetics` field in `dataset_description.json` or the\n",
    "`genetic_id` column in `participants.tsv` MUST include this file with the following\n",
    "fields:\n",
    "\n",
    "{{ MACROS___make_metadata_table(\n",
    "   {\n",
    "      \"GeneticLevel\": \"REQUIRED\",\n",
    "      \"AnalyticalApproach\": \"OPTIONAL\",\n",
    "      \"SampleOrigin\": \"REQUIRED\",\n",
    "      \"TissueOrigin\": \"OPTIONAL\",\n",
    "      \"BrainLocation\": \"OPTIONAL\",\n",
    "      \"CellType\": \"OPTIONAL\",\n",
    "   }\n",
    ") }}\n",
    "\n",
    "To ensure dataset description consistency, we recommend following [Multi-omics approaches to disease](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1215-1) by Hasin et al. 2017 to determine the `GeneticLevel:`\n",
    "\n",
    "-   `Genetic`: data report on a single genetic location (typically directly in the `participants.tsv` file)\n",
    "-   `Genomic`:  data link to participants' genome (multiple genetic locations)\n",
    "-   `Epigenomic`: data link to participants' characterization of reversible modifications of DNA\n",
    "-   `Transcriptomic`: data link to participants RNA levels\n",
    "-   `Metabolomic`: data link to participants' products of cellular metabolic functions\n",
    "-   `Proteomic`: data link to participants peptides and proteins quantification\n",
    "\n",
    "`genetic_info.json` example:\n",
    "\n",
    "```JSON\n",
    "{\n",
    "  \"GeneticLevel\": \"Genomic\",\n",
    "  \"AnalyticalApproach\": [\"Whole Genome Sequencing\", \"SNP/CNV Genotypes\"],\n",
    "  \"SampleOrigin\": \"brain\",\n",
    "  \"TissueOrigin\": \"gray matter\",\n",
    "  \"CellType\":  \"neuron\",\n",
    "  \"BrainLocation\": \"[-30 -15 10]\"\n",
    "}\n",
    "```\n",
    "\n",
    "<!-- Link Definitions -->\n",
    "\n",
    "[object]: https://www.json.org/json-en.html"
   ]
  }
 ],
 "metadata": {
  "jupytext": {
   "text_representation": {
    "extension": ".md",
    "format_name": "myst",
    "format_version": 0.13,
    "jupytext_version": "1.10.3"
   }
  },
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.13"
  },
  "source_map": [
   12
  ]
 },
 "nbformat": 4,
 "nbformat_minor": 5
}